# #FlavonoidFriday
This repository provides additional background to the #FlavonoidFriday tweets. #FlavonoidFriday is maintained by the [PuckerLab](https://www.tu-braunschweig.de/en/ifp/pbb) at TU Braunschweig.

## Introduction ##
The flavonoid biosnythesis is one of the best studied specialized metabolism pathways in plants. Flavonoids are produced by almost all land plants, but specific branches of this pathway are not present in all species. Flavonol biosynthesis, flavone biosynthesis, anthocyanin biosnythesis, and proanthocyanidin biosynthesis are some branches of the flavonoid biosynthesis. Products of these branches have different functions and the enzymes might compete for substrate when branches are active at the same time. Therefore, the gene expression of all players in this pathway is tightly regulated by a number of transcription factors (see Transcriptional Regulation). Fig. 1 provides a simplified overview of the flavonoid biosynthesis [(Pucker et al., 2020)](https://www.mdpi.com/2223-7747/9/9/1103/htm).


<a href="https://www.mdpi.com/2223-7747/9/9/1103/htm">
<img alt="Flavonoid biosynthesis pathway overview" src="https://www.mdpi.com/plants/plants-09-01103/article_deploy/html/images/plants-09-01103-g001.png" width="50%" height="50%">
</a>  


Fig. 1: Simplified overview of the flavonoid biosynthesis. (source: [Pucker et al., 2020](https://www.mdpi.com/2223-7747/9/9/1103/htm))


Flavonoids are transported from the site of biosynthesis (ER) into the central vacuole. Two potential flavonoid transport routes have been proposed: (1) transport across the tonoplast mediated by membrane-bound transport proteins and (2) vesicle-mediated transport with or without an involvement of the Golgi apparatus (Fig. 2, [Pucker & Selmar, 2022](https://www.mdpi.com/2223-7747/11/7/963/htm)).

<a href="https://www.mdpi.com/2223-7747/11/7/963/htm">
<img alt="Intracellular flavonoid transport overview" src="https://www.mdpi.com/plants/plants-11-00963/article_deploy/html/images/plants-11-00963-g002.png" width="50%" height="50%">
</a>  

Fig. 2: Simplified overview of the intracellular flavonoid transport. (source: [Pucker & Selmar, 2022](https://www.mdpi.com/2223-7747/11/7/963/htm))


### Transcriptional Regulation ###
The sophisticated regulatory network of the flavonoid biosnythesis contains members of numerous transcription factor families. Among the important families are MYBs, bHLHs, WD40s, NACs, and WRKYs. Some MYBs, bHLHs, and WD40s form a complex which is called MBW. More information about the MYBs is available through the **[MybMonday](https://github.com/bpucker/MybMonday)**.

### Mutual Exclusion of Anthocyanins and Betalains ###
While the flavonoid biosynthesis is present in (almost) all land plants, certain branches are not. One famemous example is the mutual exclusion of anthocyanins and betalains in the Caryophyllales. The yellow to red betalains are tyrosine-derived pigment group which replaced anthocyanins in several families of the Caryophyllales. Both pigments have never reported together in the same tissue and the same species. Please see [Timoneda *et al*., 2019](https://doi.org/10.1111/nph.15980) for a recent review of the complex pigment evolution in the Caryophyllales.


### Automatic annotation of the flavonoid biosynthesis ###
Do you have a new transcriptome and genome assembly of a plant species? If you are interested in studying the flavonoid biosynthesis, [KIPEs](https://github.com/bpucker/KIPEs) [(Pucker et al., 2020)](https://www.mdpi.com/2223-7747/9/9/1103/htm) could be helpful. KIPEs produces an annotation of the flavonoid biosynthesis players based on knowledge about the flavonoid biosynthesis in other species (Fig. 3). A table of the involved genes and phylogenetic trees of all players in the pathways can serve as basis to investigate the flavonoid biosynthesis in a new species. KIPEs is freely available for download via github. Additionally, we operate a web server for convenient use: [KIPEs web server](http://pbb.bot.nat.tu-bs.de/KIPEs/).

<a href="https://www.mdpi.com/2223-7747/9/9/1103/htm">
<img alt="KIPEs workflow" src="https://www.mdpi.com/plants/plants-09-01103/article_deploy/html/images/plants-09-01103-g002.png" width="50%" height="50%">
</a>  

Fig. 3: KIPEs workflow. (source: [Pucker et al., 2020](https://www.mdpi.com/2223-7747/9/9/1103/htm))


### Excellent reviews about the flavonoid biosynthesis ###
There are several excellent resources to get an overview of the flavonoid biosynthesis. This is an incomplete list:

Winkel-Shirley, 2001: Flavonoid Biosynthesis. A Colorful Model for Genetics, Biochemistry, Cell Biology, and Biotechnology. doi:[10.1104/pp.126.2.485](http://www.plantphysiol.org/content/126/2/485)

Grotewold, 2006: The Science of Flavonoids. [Book published by Springer](https://www.springer.com/de/book/9780387288215)

Tohge _et al_., 2017: Current understanding of the pathways of flavonoid biosynthesis in model and crop plants. doi:[10.1093/jxb/erx177](https://doi.org/10.1093/jxb/erx177)



## Tweets with addtional background ##

### (1) General Flavonoid Biosynthesis Overview ###
> **_<a href="https://twitter.com/boas_pucker/status/1365194891475750916">TWEET</a>:_** Flavonoids are a large group of phenylalanine-derived specialized metabolites in plants. Branches of the biosynthesis lead to flavonols, anthocyanins, proanthocyanins, flavones, and others. #FlavonoidFriday (src: https://bit.ly/2NRPlKw)

<a href="https://twitter.com/boas_pucker/status/1365194891475750916">
<img alt="Flavonoid biosynthesis pathway overview (Tweet #1)" src="https://pbs.twimg.com/media/Eu4yOCfWYAIMKxJ?format=jpg&name=4096x4096" width="20%" height="20%">
</a>

(Figure source: [Pucker et al., 2020](https://www.mdpi.com/2223-7747/9/9/1103/htm))

The background about this tweet is described above in the general introduction to the flavonoid biosynthesis.


### (2) Flavonol biosynthesis is an old branch of the flavonoid biosynthesis ###
> **_<a href="https://twitter.com/boas_pucker/status/1367746709913145345">TWEET</a>:_** Flavonols are an evolutionary old group of flavonoids. Their functions include UV & ROS protection, but also signaling & regulation of developmental processes. Flavonol production is triggered by biotic and abiotic factors #FlavonoidFriday (src: https://bit.ly/3q5Ggei)

<a href="https://twitter.com/boas_pucker/status/1365194891475750916">
<img alt="Flavonol biosynthesis pathway (Tweet #2)" src="https://pbs.twimg.com/media/EvVYRu-XUAY864Q?format=png&name=900x900" width="20%" height="20%">
</a>



### (3) Anthocyanins are colourful pigments ###
> **_<a href="https://twitter.com/boas_pucker/status/1370283419763105793">TWEET</a>:_** Anthocyanins (Greek: 'blue flower') are pigments with pH-dependent colours ranging from red to blue. Their functions include attraction of animals for pollination & seed dispersal, but also protection against abiotic stressors. #FlavonoidFriday (details: https://bit.ly/3uEZz1F)

<a href="https://twitter.com/boas_pucker/status/1370283419763105793">
<img alt="Anthocyanins (Tweet #3)" src="https://pbs.twimg.com/media/EvVbCduXMAEDII8?format=jpg&name=medium" width="30%" height="30%">
</a>


While anthocyanins are well known for their colour, it is likely that additional functions in stress response might be even more important (reviewed by Landi et al., 2015).

### (4) Proanthocyanidins are responsible for seed coat colouration ###
> **_<a href="https://twitter.com/boas_pucker/status/1372820142980272130">TWEET</a>:_** Proanthocyanidins (PA) are responsible for a dark appearance of seed coats. The biosynthesis of PAs is not completely understood and probably differs between species #FlavonoidFriday (details:https://bit.ly/38XTBjk)

<a href="https://twitter.com/boas_pucker/status/1372820142980272130">
<img alt="Proanthocyanidins (Tweet #4)" src="https://pbs.twimg.com/media/EwyzS7hWYAI6zQZ?format=jpg&name=medium" width="30%" height="30%">
</a>

LAR and ANR are involved in the PA biosynthesis in many species. However, A. thaliana lacks LAR, but is still able to produce proanthocyanidins which provide a dark colouration to the seed coat. This indicates that there must be a PA biosynthesis pathway which does not require LAR.

### (5) Dietary Flavonoids ###

> **_<a href="https://twitter.com/boas_pucker/status/1375356855279570947">TWEET</a>:_** Flavonoids are abundant phytochemicals in fruits and vegetables. Health and nutritional benefits were reported for several flavonoids including antioxidant and anti-inflammatory functions #FlavonoidFriday (details:https://bit.ly/3w3hksj)

<a href="https://twitter.com/boas_pucker/status/1375356855279570947">
<img alt="Dietary Flavonoids (Tweet #5)" src="https://pbs.twimg.com/media/ExYtlWkWEAEworj?format=jpg&name=4096x4096" width="80%" height="80%">
</a>

(Figure source: [Ahn-Jarvis et al., 2019](https://doi.org/10.3390/antiox8070202))

Flavonoids are not just responsible for beautiful flower phenotypes. These molecules can also be beneficial for human health. Flavonoids in the human diet and the potential for biomedical/biotechnological applications in the future were recently reviewed by [Ahn-Jarvis et al.](https://doi.org/10.3390/antiox8070202). They discus the releveance of increasing the bioavailability of flavonoids through agricultural engineering and targeted food design. The anti-proliferative, antioxidant, anti-inflammatory, and estrogenic activities of flavonoids could be harnessed to support classical treatments. While flavonoids show promising effects in cellular studies, the applied concentrations are almost impossible to reach with conventional food products. Changing the food processing might be one option to enhance the bioavailability. However, the bioavailability of flavonoids also depends on host factors like age and gender. Further research is required to understand the relevance of the nutrition in complex diseases.


### (6) Colours of Anthocyanins ###

> **_<a href="https://twitter.com/boas_pucker/status/1380415184741801984">TWEET</a>:_** Pelargonidin, cyanidin, peonidin, delphinidin, petunidin and malvidin are among the most common precursors of anthocyanins. #FlavonoidFriday (details: https://bit.ly/3d3BA5v)

<a href="https://twitter.com/boas_pucker/status/1380415184741801984">
<img alt="Colours of Anthocyanins (Tweet #6)" src="https://pbs.twimg.com/media/EygvdJdWEAoTWyK?format=jpg&name=900x900" width="40%" height="40%">
</a>


Pelargonidin (Pg), cyanidin (Cy), peonidin (Pn), delphinidin (Dp), petunidin (Pt) and malvidin (Mv) are the six most common anthocyanidins ([Sun et al., 2012](https://onlinelibrary.wiley.com/doi/full/10.1002/rcm.6209)). Anthocyanidins are unstable until sugar groups are attached. This modification step converts anthocyanidins (aglycones) into anthcyanins. Anthocyanins are contribute to a wide range of colours ranging from red to mangenta (cyanin), from orange to red (pelargonin), and from magenta to purple (delphinin) ([Ferreyra et al., 2012](https://doi.org/10.3389/fpls.2012.00222)).


### (7) Replacement of Anthocyanins by Betalains ###

> **_<a href="https://twitter.com/boas_pucker/status/1382951896089038849">TWEET</a>:_** While anthocyanins are present in most plant species, they are replaced by another group of pigments in some families of the Caryophyllales: betalains. #FlavonoidFriday
(src:https://doi.org/10.1111/nph.15980)

<a href="https://twitter.com/boas_pucker/status/1382951896089038849">
<img alt="Replacement of Anthocyanins by Betalains (Tweet #7)" src="https://pbs.twimg.com/media/EzEFv8lWEAAoA3H?format=jpg&name=large" width="30%" height="30%">
</a>


Both pigments (anthocyanins and betalains) were never observed together in the same plant which resulted in the mutual exclusion theory. A molecular mechanism which could cause this mutual exclusion is not known yet. Betalains do not occur in plant species outside the Caryophyllales. The current knowledge about the evolution of the betalain biosynthesis was recently reviewed in [Timoneda et al., 2019](https://doi.org/10.1111/nph.15980).


### (8) Anthocyanins - more than just colourful ###

> **_<a href="https://twitter.com/boas_pucker/status/1385473515101503489">TWEET</a>:_** Anthocyanins are not just colourful pigments, but have additional funtions: 1) metal-chelating agent 2) photoprotection & antioxidant 3) modulator of ROS signaling 4) delay of senescence #FlavonoidFriday (details: https://bit.ly/32Fvoup)

<a href="https://twitter.com/boas_pucker/status/1385473515101503489">
<img alt="Anthocyanins - more than just colourful (Tweet #8)" src="https://pbs.twimg.com/media/Ezou1oRWQAApjzb?format=png&name=900x900" width="30%" height="30%">
</a>


Anthocyanins are well known for the colouring of flowers and other plant tissue. However, the evolution of these pigments might not only be due to the colour. It is possible that additional functions provided evolutionary advantages. A publication by [Landi et al., 2015](https://doi.org/10.1016/j.envexpbot.2015.05.012) describes some of these functions:
1) Anthocyanins can serve as metal-chelating agents.
2) Anthocyanins can have a photoprotective and antioxidant effect.
3) The ROS signaling can be modulated by anthocyanins.
4) It seems that anthocyanins can delay senescencs that is promoted by nutient deficiency.
The functions and possible also additional ones can contribute differently to the evolutionary value of anthocyanins in numerous land plant species.


### (9) Red leaf colour of _Perilla frutescens_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1388002675455782912">TWEET</a>:_** A relative increase of anthocyanin biosynthesis over flavonol biosynthesis seems to explain the leaf colour difference between the Perilla frutescens cultivars JIZI1 and JIZI2. #FlavonoidFriday (details: https://bit.ly/3eAA1ve)


<a href="https://twitter.com/boas_pucker/status/1388002675455782912">
<img alt="Red leaf colour of _Perilla frutescens_ (Tweet #9)" src="https://pbs.twimg.com/media/E0MY1HHWQAgGKgN?format=png&name=900x900" width="30%" height="30%">
</a>


_Perilla frutescens_ is a Chinese plant with nutritional and medical use. Flavonoids are probably an important factor. The varieties JIZI1 and JIZI2 differ in the (relative) amount of accumulated flavonoids. The green vs. red leaf colour difference indicates that anthocyanins might be involved. Differential expression analysis revealed 15 enzymes involved in this pathway. [Jiang et al., 2020](https://doi.org/10.1038/s41598-020-73274-y) state that 109 flavones, 33 flavonols, 27 flavonoids, 22 anthocyanins, 20 flavanones, and 12 isoflavones differ in their concentration between both cultivars. A gene expression analysis of the flavonoid biosynthesis shows up-regulation of almost all steps except for CHI and FLS. This might suggest (1) a general up-regulation of the pathway and (2) that the flux is directed towards anthocyanins and away from flavonols.

However, several open questions remain:
1) Why are DFR and ANS not up-regulated on all parallel branches of the flavonoid biosynthesis?
2) Is UGT75C1 accepting flavonol derivates and not just anthocyanins?
3) Which sequences are the orthologs of well characterized transcriptional regulators like MYB75, TT8, and TTG1?
4) Is the transcriptome assembly available?



### (10) Decoration of anthocyanins differs between species ###

> **_<a href="https://twitter.com/boas_pucker/status/1390562044575850497">TWEET</a>:_** Anthocyanins are modified by glycosyltransferases, methyltransferases, and acyltransferases. These modifications can stabilize anthocyanins and are responsible for the huge diversity. #FlavonoidFriday (details: https://bit.ly/2QZPGww)

<a href="https://twitter.com/boas_pucker/status/1390562044575850497">
<img alt="Decoration of anthocyanins differs between species (Tweet #10)" src="https://pbs.twimg.com/media/E0xCO0CXEAEA9tq?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

[Tohge et al., 2017](https://doi.org/10.1093/jxb/erx177) provide a great summary of the current knowledge about the diversity of the flavonoid decoration/modification steps. While the aglycon biosynthesis is well characterized and highly conserved between species, the modification appears to be more flexible. Generally, sugars can be added at the OH groups at the C3, C5, and C7 positions. Not just single sugars, but also sugar chains or trees can serve as decoration. A recently identified flavonol modifying enzyme is BGLU6 ([Ishihara et al., 2016](https://doi.org/10.1093/jxb/erv546)) which shows differences between A. thaliana accessions i.e. within a species. Therefore, it is not surprising that these decoration pathways are different between plant species.



### (11) Repressors of the anthocyanin biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1393091204762402816">TWEET</a>:_** Repressors of the anthocyanin biosynthesis are regulated by (1) development, (2) environmental triggers, or (3) plant hormones. Proteins and small RNAs can play a role. #FlavonoidFriday (details:https://bit.ly/3w8cfhg)



<a href="https://twitter.com/boas_pucker/status/1390562044575850497">
<img alt="Repressors of the anthocyanin biosynthesis (Tweet #11)" src="https://pbs.twimg.com/media/E1U6b3VX0AAL3o1?format=jpg&name=4096x4096" width="30%" height="30%">
</a>


#### Negative regulators of the anthocyanin biosynthesis ####

An excellent Tansley review by [LaFountain & Yuan (2021)](https://doi.org/10.1111/nph.17397) summarizes the current knowlege about negative regulators of the anthocyanin biosynthesis and gives perspective for future research. Anthocyanin repressors are generally activated by (1) developmental processes, (2) environmental factors, or (3) plant hormones. Most repressors destablize the MBW complex which would otherwise activate anthocyanin biosynthesis genes. While the first anthocyanin repressors were identified more than 20 years ago, most were discovered in the last decade. Activators of the anthocyanin biosynthesis belong to R2R3-MYB subgroup 6 or 5. In contrast, many repressors fall in subgroup 4.
1) There are active repressors in subgroup 4, which can be classified into FaMYB1-type and AtMYB4-type. It appears that an EAR motif is crucial for the repressor function. LaFountain & Yuan describe the function of the repressors as a "break" instead of an "off-switch". 
2) Another group of repressors lost one MYB repeat and is characterized by the presence of a 'TLLLFR' motif. There are indications for multiple different origins of these anthocyanin biosynthesis repressors.
3) The CPC-clade of repressors emerged in the common ancestor of monocots and dicots. Members of this clade lack the bHLH interaction motif which leads to passive represssion by competing for the bHLH component of the MBW complex. Expression of these MYBs is activated by the MBW complex which creates a periodic pattern of gene expression which becomes visible as spots or stripes of pigmentation.
4) miR156-SPL is highly expressed in seedlings and down-regulated a negative regulator (_SQUAMOSA PROMOTER BINDING PROTEIN-LIKE 9_) of the anthocyanin biosynthesis. This explains a temporal pigmentation of seedlings called "juvenile reddening" which is lost in later stages.
5) miR858 targets the anthocanin activator MYB113 in A. thliana. miR828 targets the negative regulator _MYBL2_ in _A. thaliana_, but the positive regulator _MYB7-like_ in tomato.
6) GL2 in _A. thaliana_ is a HD-ZIP family member that down-regulates _PAP2_ and _MYB113_. HAT1 is another HD-ZIP member that represses anthocyanin biosynthesis. GL2 and HAT1 might be a lineage-specific specialty of _A. thaliana_. In contrast, HB1 (discovered in apple) might be a more conserved repressor of the HD-ZIP family.


### (12) MYB117 can boost the anthocyanin biosynthesis in poplar ###

> **_<a href="https://twitter.com/boas_pucker/status/1395620374864302083">TWEET</a>:_** Overexpression of poplar MYB117 boosts the anthocyanin biosynthesis. Up-regulation of the flavonoid 3',5'-hydroxylase genes leads to increased B-ring hydroxylation of flavonoids. #FlavonoidFriday (details: https://bit.ly/3wlSlzv)

<a href="https://twitter.com/boas_pucker/status/1395620374864302083">
<img alt="MYB117 can boost the anthocyanin biosynthesis in poplar (Tweet #12)" src="https://pbs.twimg.com/media/E144mTUXEAI8rij?format=jpg&name=medium" width="30%" height="30%">
</a>

[Ma et al., 2021](https://academic.oup.com/jxb/article/72/10/3864/6169232?login=true) present the poplar MYB117 as a powerful activator of the anthocyanin biosynthesis. Overexpression under control of the 35S promoters provides a general boost to the anthocyanin biosynthesis. The up-regulation of F3'5'H genes causes an increased B-ring hydroxylation which affects also other branches of the flavonoid biosynthesis. Multiple investigations of several flavonoid biosynthesis activators suggest that different MYBs can fine tune the hydroxylation level of flavonoids. MYB117 sits in an array of six PAP1-type MYBs: MYB113, MYB116, MYB117, MYB118, MYB119, and MYB120.


### (13) Tobacco CHS is crucial for salt tolerance ###

> **_<a href="https://twitter.com/boas_pucker/status/1398172184602943491">TWEET</a>:_** Overexpression of the tobacco MYB4 reduces the salt tolerance of seedlings through repression of flavonoid production. The flavonoid biosynthesis is inhibited by down-regulation of the CHS (first step) #FlavonoidFriday (details: https://bit.ly/2TfMcqm)


<a href="https://twitter.com/boas_pucker/status/1398172184602943491">
<img alt="Tobacco CHS is crucial for salt tolerance (Tweet #13)" src="https://pbs.twimg.com/media/E2dFaAeXoAAUdjK?format=png&name=small" width="30%" height="30%">
</a>

Salt stress is known to induce the flavonoid biosynthesis. [Chen et al. 2019](https://doi.org/10.3389/fpls.2019.00178) report that rutin content increased in tobacco under salt stress. However, the overexpression of NtMYB4 caused a reduced production of rutin leading to a lower salt tolerance. MYB4 overexpression lines were more similar to the WT when rutin was externally supplied. Rutin is probably involved in reduction of ROS which accumulates under salt stress. It seems that NtMYB is a strong negative regulator of the CHS, but does not affect ANS substantially. However, down-regulation of CHS by RNAi lead to a reduced transcript abundance of the downstream genes.


### (14) TTG2 controls epicatechin transport ###

> **_<a href="https://twitter.com/boas_pucker/status/1400693803057438724">TWEET</a>:_** TTG2 (WRKY44) influences the seed coat pigmentation through regulation of the epicatechin transport (TT12, AHA10). Formation of a MBW-like complex comprising TTG2+TTG1+TT8 was postulated #FlavonoidFriday (details: https://bit.ly/3wPIK4a)


<a href="https://twitter.com/boas_pucker/status/1400693803057438724">
<img alt="TTG2 controls epicatechin transport (Tweet #14)" src="https://pbs.twimg.com/media/E2xycrXWQAEfcLg?format=png&name=900x900" width="30%" height="30%">
</a>

[Gonzalez et al., 2016](https://doi.org/10.1016/j.ydbio.2016.03.031) present TTG2 as an important regulator of epicatechin transporters like TT12/MATE and TT13/AHA10. Epicatechins are imported into the vacuoles of seed coats where they are condensed into proanthocyanidins (condensed tannins). Proanthocyanidins (PAs) provide the dark colouration to seeds. There is a large mutant collection comprising lines with specific blocks in the PA biosynthesis or accumulation pathway. These are called transparent testa (tt) mutants [Appelhagen et al., 2014](https://doi.org/10.1007/s00425-014-2088-0). A transcription factor complex of MYB+bHLH+WDR is known to regulate several steps in the flavonoid biosynthesis and especially in the anthocyanin and PA biosynthesis. _TTG2_ (_WRKY44_) expression is controlled by the MBW complex [Ishida et al., 2007](https://doi.org/10.1105/tpc.107.052274). TTG2 shows high binding affinity towards the W box, a motif of (C/T)TGAC(T/C), which was detected in the promoter sequences of many target genes. TTG2 is specific to the PA biosynthesis, because anthocyanins are not affected in the ttg2 mutant. The TT12 promoter region contains several W boxes, but it is not cleary which of them is the actual TTG2 binding site. TT12 overexpression partialy rescued the ttg2 phenotype thus suggesting that this is one of the most important targets. In summary, TTG2 appears to be a regulator of the PA transport pathway.


### (15) Role of TT10 in the seed coloration ###

> **_<a href="https://twitter.com/boas_pucker/status/1403230516552237056">TWEET</a>:_** TT10 is a laccase-like polyphenol oxidase which appears to be involved in the oxidative polymerization of flavonoids in developing Arabidopsis thaliana seed coats. The TT10 products are responsible for the brown seed color. #FlavonoidFriday (details: https://bit.ly/34KmA7u)


<a href="https://twitter.com/boas_pucker/status/1403230516552237056">
<img alt="Role of TT10 in the seed coloration (Tweet #15)" src="https://pbs.twimg.com/media/E286633XoAQxJZC?format=jpg&name=large" width="30%" height="30%">
</a>


[Pourcel et al., 2005](https://dx.doi.org/10.1105%2Ftpc.105.035154) characterized _TRANSPARENT TESTA 10_ (_TT10_) which encodes a laccase-like enzymes which is most likely involved in the oxidative polymerization of epicatechins and catechins in the seed coat of _A. thaliana_. These oxidized compounds provide the brown coloration of seeds. Oxidized forms of colorless epicatechins and catechins are called quinones which turn from yellow to brown with increase in oxidation level. Given the TT10 function, it is not surprising that the _tt10_ mutant shows a delayed appearance of the brown color. None of the other 16 laccase-like proteins in _A. thaliana_ are able to fullfill the TT10 function. Instead an autoxidation is assumed in the _tt10_ mutant. Epicatechin and catechin accumulate in the vacuole, but might be released into the cytoplasm upon cell death through a burst of the vacuole. TT10 would get in contact with its substrates and could produce oxidized and polymerized products. Additionally, TT10 might process PAs in the cell wall. TT10 is also involved in the formation of flavonol dimers (quercetine rhamnoside oxidative polymerization), but this product does not contribute to the brown seed color. Interestingly, it appears that _TT10_ is not regulated by the TT2+TT8+TTG1 complex like other genes in the proanthocyanidin biosynthesis.



### (16) Phenylpropanoids in strawberry ###

> **_<a href="https://twitter.com/boas_pucker/status/1405767234367234048">TWEET</a>:_** F3'H seems to be a decisive step in the flavonoid biosynthesis of strawberry, because higher gene expression correlates with increased content of specific flavonoids #FlavonoidFriday (details: https://bit.ly/3vw4VeZ)


<a href="https://twitter.com/boas_pucker/status/1405767234367234048">
<img alt="Phenylpropanoids in strawberry (Tweet #16)" src="https://pbs.twimg.com/media/E4IpNXlVkAAfp2b?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

Flavonoids have nutritional benefits for humans mostly due to their antioxidant activity. [Pott et al., 2020](https://www.nature.com/articles/s41598-020-76946-x) investigated the flavonoid biosynthesis as well as the general phenylpropanoid biosynthesis in strawberry. The composition and concentration of flavonoids changes during strawberry development: tannins are produced in the early stages and flavonols and anthocyanins in the later stages. Important for the pigmentation of the fruits are glycosylated pelargonidins and cyanidins. The authors investigated the heritability of phenylpropanoid metabolites including flavonoids. QTL for phenylpropanoids were identified, but there is a large number of QTL spanning substantial parts of almost all linkage groups. However, there are hotspots which seem to be associated with multiple phenlypropanoids. An interesting observation is that one copy of the anthocyanin repressor FaMYB was located in a QTL, but the gene expression does not correlate with the investigated flavonoids. The authors present F3'H as a potentially decisive step in the flavonoid biosynthesis that can control the metabolic flux. The correlation of F3'H gene expression correlates with the amount of specific flavonoiods (see paper for details). The substrate specificity of enzymes with respect to hydroxylation patterns needs further investigation.


### (17) MYB4, MYB7, and MYB32 are anthocyanin repressors ###

> **_<a href="https://twitter.com/boas_pucker/status/1408334154614247425">TWEET</a>:_** MED5a & MED5b control the expression of AtMYB4 which interacts with the bHLHs TT8, GL3, and EGL3 leading to a reduced expression of anthocyanin and proanthocyanidin biosynthesis genes #FlavonoidFriday (details: https://bit.ly/3qqsMM1)


<a href="https://twitter.com/boas_pucker/status/1408334154614247425">
<img alt="MYB4, MYB7, and MYB32 are anthocyanin repressors (Tweet #17)" src="https://pbs.twimg.com/media/E4tj1ieX0AU-4cw?format=jpg&name=large" width="40%" height="40%">
</a>


[Wang et al., 2019](https://doi.org/10.1111/tpj.14570) report that MYB4, MYB7, and MYB32 interact with the bHLHs TT8, GL3, and EGL3 leading to a repression of the flavonoid biosynthesis. MYB4 is also a direct repressor of Arogenate Dehydratase 6 (ADT6) - the last step in the shikimate biosynthesis, which feeds substrate to the flavonoid biosynthesis. A complex of MED5a and MED5b regulates the gene expression of MYB4, MYB7, and MYB32. If only MED5a is knocked-out, the anthocyanin accumulation is increased due to a reduced expression of the repressors MYB4, MYB7, and MYB32. While MYB4, MYB7, and MYB32 can bind to bHLHs, they do not interact with the anthoyanin activating MYBs. MYB4, MYB7, and MYB32 show different affinities for specific bHLHs suggesting a fine regulation of specific MBW complexes.



### (18) Flavonoid biosynthesis regulator miR858 ###

> **_<a href="https://twitter.com/boas_pucker/status/1410855763678011398">TWEET</a>:_** pri-miR858a encodes a small peptide which promotes miR858 transcription. miR858a controls lignin and flavonoid biosynthesis through repression of MYB12 #FlavonoidFriday (details: https://bit.ly/3hl2rL6)

<a href="https://twitter.com/boas_pucker/status/1410855763678011398">
<img alt="Flavonoid biosynthesis regulator miR858 (Tweet #18)" src="https://pbs.twimg.com/media/E5RYxV0WQAEHYLN?format=jpg&name=large" width="40%" height="40%">
</a>

miRNAs are single stranded RNAs that regulate gene expression through increased transcript degradation or repressed translation. These miRNAs can encode small proteins which increase their own transcription. miR858 is a repressor of the MYB11/MYB12/MYB111 transcription. This negative regulation results in reduced flavonol production. [Sharma et al., 2016](https://pubmed.ncbi.nlm.nih.gov/27208307/), [Wang et al., 2016](https://pubmed.ncbi.nlm.nih.gov/27450422/), and [Sharma et al., 2020](https://www.nature.com/articles/s41477-020-00769-x) investigated the functions of miR858. The latest study used a modification via (1) CRISPR/Cas-based modification and (2) supplementation of miPEP858a.
miPEP858a was identified through the test of all potential ORFs in a GUS fusion experiment. Only one of the potential ORFs was actually translated leading to detectable GUS activity.
Supplementation of miPEP585a in the media resulted in enhanced transcription of miR858 and slightly shorter roots. Several additional experiments support that miPEP858a is important in the transcriptional control of miR858.
Overexpression of miR858 resulted in reduced flavonol and anthocyanin aglycones as well as increased lignin levels. The modification (knock-out) of miR858 resulted in the opposite effect i.e. increased anthocyanin and flavonol levels at the cost of lignin.
It is known that flavonoids have a negative impact on auxin transportation and plant growth. Loss of miR858 leads to lower expression of auxin-associated genes, while the overexpression of miR858 increases the expression of these genes.


### (19) Anthocyanin degradation ###

> **_<a href="https://twitter.com/boas_pucker/status/1413384926473379840">TWEET</a>:_** Accumulation of anthocyanins in the central vacuole leads to a coloration of plant structures. Numerous modifications and degradation determine how quickly the color fades #FlavonoidFriday (details: https://bit.ly/3yCpurZ)


<a href="https://twitter.com/boas_pucker/status/1413384926473379840">
<img alt="Anthocyanin degradation (Tweet #19)" src="https://pbs.twimg.com/media/E50YKbgWUAAE8DW?format=jpg&name=medium" width="40%" height="40%">
</a>


[Passeri et al., 2016](https://doi.org/10.3389/fpls.2016.00153) reviewed the stabilization of anthocyanins in the vacuole with respect to biotechnological applications. Enriching food with anthocyanins is important, because several studies identified health promoting effects of anthocyanins. Generating new colors through modifications of anthocyanins is of huge importance for the ornamental plant and cut flower industry.
Anthocyanin decorating enzymes include UDP-glucose:flavonoid 3-O-glucosyltransferase (3UFGT), rhamnosyl transferase (RT), UDP-glucose:flavonoid 5-O-glucosyltransferase (5UFGT), anthocyanin acyltransferase (AAT), and methyltransferase (MT). Expression of the corresponding genes is controlled by the MBW complex. Loss of anthocyanin pigmentation is often caused by mutations in the regulator genes (MYB, bHLH, WDR), because mutations in structural genes would be insufficient or could have pleitropic effects.
Active degradation of anthocyanins could be catalyzed by polyphenol oxidases if the glycosylation is missing/lost. The relevance of different anthocyanin decorations on degrading enzymes is still under investigation. This anthocyanin degradation can strongly affect the market value of ornamental plants. Therefore, it is important to understand the process and to improve plants bassed on this knowledge. A vacuolar peroxidase is suspected to be involved in this anthocyanin degradation process. A proton pump (AHA10) influences the pH value of the vacuole, but this has only a minor effect on anthocyanin fading if any. The anthocyanin degradation in leaves might be beneficial for photosynthesis efficiency. The degradation in flowers is coupled with the release of fragrant volatils which could be a signal for pollinators.



### (20) Mutual exclusion of anthocyanins and betalains in _Hylocereus_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1415921640060297219">TWEET</a>:_** Anthocyanins are replaced by betalains in some Caryophyllales. There is a mutual exclusion of both pigments. A species is only anthocyanin-pigmented OR betalain-pigmented #FlavonoidFriday (https://bit.ly/3xGuRXf & https://rdcu.be/cn0UW)

<a href="https://twitter.com/boas_pucker/status/1415921640060297219">
<img alt="Mutual exclusion of anthocyanins and betalains in _Hylocereus_ (Tweet #20)" src="https://pbs.twimg.com/media/E50laJ-XsAMoet0?format=jpg&name=medium" width="40%" height="40%">
</a>

A recent publication by Fan et al., 2020 in BMC Plant Biology claimed the detection of anthocyanins in the betalain-pigmented genus _Hylocereus_ and stated that anthocyanins are responsible for the red coloration of pitaya fruits. This analysis was based on the comparison of a white, a pink, and a red pitaya cultivar. However, this paper was retracted due to various issues. A [re-analysis](https://doi.org/10.1186/s12870-021-03080-9) of the RNA-seq data sets revealed that transcripts of the anthocyanin biosynthesis genes (_DFR_ & _ANS_) are barely detectable. Moreover, the abundances of these transcripts in different cultivars do not support the corresponding pigment phenotypes. Several studies already reported that betalains are responsible for the red coloration of pitaya fruits. In summary, the re-analysis supports the mutual exclusion paradigm and suggests that not anthocyanin, but betalains, are responsible for the coloration of pitaya fruits.



### (21) KIPEs - automatic identification of flavonoid biosynthesis genes or enzymes ###

> **_<a href="https://twitter.com/boas_pucker/status/1418443256405262345">TWEET</a>:_** Looking for flavonoid biosynthesis genes/enzymes in a transcriptome or genome assembly? KIPEs can perform this identification automatically in your species of interest #FlavonoidFriday (https://bit.ly/3fpykTw & https://bit.ly/3ikpFlh)

<a href="https://twitter.com/boas_pucker/status/1418443256405262345">
<img alt="KIPEs - automatic identification of flavonoid biosynthesis genes or enzymes (Tweet #21)" src="https://pbs.twimg.com/media/Eu4yOCfWYAIMKxJ?format=jpg&name=4096x4096" width="20%" height="20%">
</a>

[KIPEs](https://github.com/bpucker/KIPEs) is freely available via github. Data sets are included to allow the identification of genes/enzymes of the core flavonoid biosynthesis pathway. Additional data sets to find the players of additional reactions are under construction. More pathways will be supported in the future. Please get in touch if you have suggestions for pathways that should be included.



### (22) Brassinosteroids inhibit flavonoid biosynthesis in apple ###

> **_<a href="https://twitter.com/boas_pucker/status/1421017725330149377">TWEET</a>:_** Brassinosteroid treatment reduces the flavonoid biosynthesis in apple. BR-induced MdBEH2.2 might form a complex with the anthocyanin regulator MdMYB60 leading to this inhibition #FlavonoidFriday (details: https://bit.ly/3x7GVQ4)

<a href="https://twitter.com/boas_pucker/status/1421017725330149377">
<img alt="Brassinosteroids inhibit flavonoid biosynthesis in apple (Tweet #22)" src="https://pbs.twimg.com/media/E7hyOQPXMAE8pLf?format=jpg&name=medium" width="20%" height="20%">
</a>

[Wang et al., 2021](https://doi.org/10.1093/jxb/erab284) investigated the inhibitation of the flavonoid biosynthesis in red-fleshed apple by brassinosteroids. Brassinosteroids (BRs) are steroid hormones with an inibitory effect on the flavonoid biosynthesis. A central inhibitor involved in this process is probably MdBEH2.2 which form a coplex with MdMYB60 leading to an inhibition of this MYB protein. However, many additional proteins are involves in detecting BRs and transmitting the triggered singnal. While treatments with BRs lead to a reduced flavonoid accumulation, the opposite effect was observed in treatments with a BR biosynthesis inhibitor (brassinazole).


### (23) Auxiliary proteins in the flavonoid biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1423531785556664322">TWEET</a>:_** Chalcone isomerase-like (CHIL) and pathogenesis-related 10 (PR10) proteins play probably non-enzymatic, but important regulatory roles in the flavonoid biosynthesis #FlavonoidFriday (details: https://bit.ly/3fBuPsw)

<a href="https://twitter.com/boas_pucker/status/1423531785556664322">
<img alt="Auxiliary proteins in the flavonoid biosynthesis (Tweet #23)" src="https://pbs.twimg.com/media/E8Fjt8NWYAAELgq?format=jpg&name=4096x4096" width="20%" height="20%">
</a>


Two families of auxiliary proteins are important regulators of the flavonoid biosynthesis ([Dastmalchi, 2021](https://onlinelibrary.wiley.com/doi/epdf/10.1111/tpj.15446)):
(1) Chalcone isomerase-like (CHIL) proteins bind to CHS and influence the catalyzed reaction. This might be necessary to reduce the promiscuity of CHS, which was observed in _in vitro_ assays. CHILs increased the flavonoid biosynthesis in plants. Loss of this activity can lead to a loss of pigmentation in flower and fruit. These CHILs are closely related to _bona fide_ CHIs and form the type IV CHI clade. Members of the type III CHIs have also been reported to act as polyketide-binding proteins (PBPs). These non-CHI proteins lack the amino acids of the CHI active center and position the ligand naringenin in a non-productive. However, it appears that CHILs are crucial for the coordination between CHS and CHI. It is possible that CHILs influence the biosynthesis of different anthocyanins, but do not alter the overall amount. The R2R3-MYB TT2 was identified as regulator of _A. thaliana_ CHIL.

(2) Pathogenesis-related 10 (PR10) proteins seem to bind intermediates of the flavonoid biosynthesis and could have an impact on the anthocyanin composition. PR10 is also involved in other spececialized metabolism pathways. Unfortunately, there is no uniform nomenclature for members of this gene family, because they were discovered and described independently in different species. PR1 in _Petroselium crispum_, _Bet v 1_ in _Betula verrucosa_, and Major latex proteins (MLPs) in _Papaver somniferum_. The PR10 clade is highly susceptible to neofunctionalization.


### (24) Flavonoid absorption spectra ###

> **_<a href="https://twitter.com/boas_pucker/status/1426076051352875013">TWEET</a>:_** Flavonoid absorption spectra depend on the pH. Cyanidin- and pelargonidin-based anthocyanins peak in blue-green wavelength at low pH, but shift to longer wavelengths at higher pH #FlavonoidFriday (details: https://bit.ly/37F7PVa)

<a href="https://twitter.com/boas_pucker/status/1426076051352875013">
<img alt="Flavonoid absorption spectra (Tweet #24)" src="https://pbs.twimg.com/media/E8pkBfmWEAQvU1p?format=jpg&name=large" width="20%" height="20%">
</a>


[Stavenga et al., 2021](https://doi.org/10.3389/fpls.2020.600124) investigated the impact of pH change on the absorption spectra of extracts from flowers. The color of flavonoids accumulating in the central vacuole is generally dependent on the vacuolar pH. Investigated species are the red flowering _Papaver dubium_, the orange flowering _Meconopsis cambrica_, and two _Mandevilla sanderi_ varities that flower in red and white, respectively. The absorption spectra of red extracts from _P. dubium_ flowers can be explained by cyanidin and pelargonidin glycosides. An analysis of _P. rhoeas_ extracts revealed that the contribution of flavonols with the same hydroxylation pattern (kaempferol and quercetin) is rather small. White flowers of _M. sanderi_ show a strong absorption in the UV range indicating the presence of flavonols. Methanolic extracts represent the native plant tissue well. 



### (25) Petal color differences in _Lysimachia arvensis_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1428605215377289218">TWEET</a>:_** DFR and F3'5'H are two important genes influencing the petal color of Lysimachia arvensis. Two DFR copies differ in amino acid residues which probably cause substrate affinity changes #FlavonoidFriday (details: https://bit.ly/3ghF2e1)


<a href="https://twitter.com/boas_pucker/status/1428605215377289218">
<img alt="Petal color differences in _Lysimachia arvensis_ (Tweet #25)" src="https://pbs.twimg.com/media/E9NpifSXEAAhV8b?format=jpg&name=4096x4096" width="20%" height="20%">
</a>


[Sanchez-Cabera et al.](https://www.frontiersin.org/articles/10.3389/fpls.2021.633979/full) investigated the flower color transition from blue to orange in _Lysimachia arvensis_ based on a transcriptomic data sets. Blue and orange flowered _L. arvensis_ plants can be found across Europe. Interestingly, the proportion of blue-flowered plants increases with temperature and daylength, while higher precipitation leads to a decrease. 39 of all 140 genes associated with the flavonoid biosynthesis showed differential expression. _F3'5'H_ and _DFR_ were among these DEGs. _F3'5'H_ showed reduced expression in orange petals and two DFR copies revealed differential expression. These DFR copies showed amino acid substitutions that affected the substrate specificity region: primarily malvidin (blue) to primarily pelargonidin (orange). A phylogenetic analysis revealed that these DFR copies belong to two distinct lineages that resulted from a duplication deep in the angiosperms. Another important factor for the orange coloration is probably the high abundance of BZ1-2 which adds a glucose to the 3-position of anthocyanidins. Blue petals showed high abundance of OMT (methylation of delphinidin) and 3GT (adds rhamnoside), which are probably required for stabilization of the pigments. 



### (26) Flavonoid biosynthesis in _Cosmos caudatus_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1431134381549113345">TWEET</a>:_** Microscopic analysis of anthocyanin patterns in _Cosmos caudatus_ reveals pigmentation of individual cells in the epidermis #FlavonoidFriday (details: https://bit.ly/3ksUGV7)


<a href="https://twitter.com/boas_pucker/status/1431134381549113345">
<img alt="Flavonoid biosynthesis in _Cosmos caudatus_ (Tweet #26)" src="https://pbs.twimg.com/media/E9xFDprXsAQ8pet?format=jpg&name=large" width="20%" height="20%">
</a>


[Gunasekaran et al., 2021](https://doi.org/10.3390/agronomy11040661) looked at the anthocyanin biosynthesis in _Cosmos caudatus_ with omics and microscopic methods. The microscopic analysis revealed strong pigmentation of individual cells in the epidermis of different plant parts. It would be very interesting to understand how this pigmentation pattern emerges. For example, it would be exciting to see if the F3'5'H expression correlates with the blue pigmentation. A connection of the metabolite levels (Fig.7) with the gene expression in the same tissue could help to answer such questions.

BLAST does not perform a search for "homology", but only identifies similar sequences which may or may not be homologs. While the best BLAST hits are often orthologs, the results can be misleading in some cases. Therefore, it is better to rely on global alignments and a phylogenetic analysis. This would have been a perfect case for [KIPEs](https://www.mdpi.com/2223-7747/9/9/1103/htm) to perform an automatic annotation and to find the actual functional isoform among the up to 12 unigenes for steps in the flavonoid biosynthesis. A phylogenetic tree could help to visually distinguish between functional and non-functional isoforms.




### (27) Red poplar leaves ###

> **_<a href="https://twitter.com/boas_pucker/status/1433671098772508672">TWEET</a>:_** Color differences of poplar leaves involve an up-regulation of HY5, HYH, and TTG2 which activate the anthocyanin biosynthesis #FlavonoidFriday (details: https://bit.ly/3B8afIC)

<a href="https://twitter.com/boas_pucker/status/1433671098772508672">
<img alt="Red poplar leaves (Tweet #27)" src="https://pbs.twimg.com/media/E9x2l17XEAA7WsG?format=jpg&name=large" width="20%" height="20%">
</a>

[Chen et al., 2021](https://doi.org/10.1007/s11103-021-01166-4) investigated the molecular mechanisms underlying the color differences of poplar leaves. Analyses on the transcriptomic and proteomic level were combined. The transcription factors HY5, HYH, and TTG2 appeared to be involved in the up-regulation of anthocyanin biosynthesis in red leaves. Anthocyanins are presented as the major factor responsible for the color change from green to red. There seems to be a negative correlation between the chlorophyll and the anthocyanin content in some cases. 

While this study provides first insights, there are also several open questions:
(1) Although it seems that anthocyanins are contributing to this color differences, it is not clear if additional factors are involved.
(2) It seems that genes for some steps in the pathway were not identified in one or the other species (Fig. 5). The missing steps are C4H, 4CL, and ANS. This is surprising as all three would be required for the anthocyanin biosynthesis.
(3) What is about the GM/MT/AT and GST steps whcih are apparently not covered by any gene/protein?




### (28) Radish GST involved in anthocyanin transport ###

> **_<a href="https://twitter.com/boas_pucker/status/1436215360923996170">TWEET</a>:_** RsGST1 is involved in anthocyanin and proanthocyanidin accumulation in radish #FlavonoidFriday (details: https://bit.ly/3kYMxIq)

<a href="https://twitter.com/boas_pucker/status/1436215360923996170">
<img alt="Radish GST involved in anthocyanin transport (Tweet #28)" src="https://pbs.twimg.com/media/E-qxzOhWUAEbS6E?format=jpg&name=large" width="20%" height="20%">
</a>

[Lai et al., 2021](https://pubmed.ncbi.nlm.nih.gov/34247029/) identified and characterized RsGST1 in radish which can complement _Arabidopsis thaliana tt19_ (_gstf12_) mutants. This indicates that this gluthathione S-transferase is involved in the anthocyanin accumulation in the vacuole. RsGST1 was identified from a set of 40 GST members in radish. Additional evidence came from a higher expression of _RsGST1_ in red tissues compared to white ones. Overexpression of _RsGST1_ alone does not increase the anthocyanin biosynthesis. Overexpression of _RsGST1_ and the anthocyanin activating _RsMYB1a_ increased anthocyanin production and accumulation. It was also demonstrated that RsMYB1a is a transcriptional activator of _RsGST1_.



### (29) Artificial producation of anthocyanins in the betalain-pigmented cactus ###

> **_<a href="https://twitter.com/boas_pucker/status/1438752075664482304">TWEET</a>:_** Artificial production of anthocyanins in the betalain-pigmented cactus Astrophytum myriostigma through heterologous expression of anthocyanin biosynthesis genes #FlavonoidFriday (details: https://bit.ly/2WU1ZNu)

<a href="https://twitter.com/boas_pucker/status/1438752075664482304">
<img alt="Artificial producation of anthocyanins in the betalain-pigmented cactus (Tweet #29)" src="https://pbs.twimg.com/media/E-q5QKBXMAAo5xZ?format=jpg&name=large" width="40%" height="40%">
</a>

[Sakuta et al., 2021](https://doi.org/10.1007/s10265-021-01341-0) achieved the artificial production of anthocyanins in a betalain-pigmented species. Naturally, anthocyanins and betalains are mutually exclusive (they do not co-occur in the same plant species) as recently reviewed [Timoneda et al., 2019](https://doi.org/10.1111/nph.15980). Overexpression of the anthocyanin biosynthesis genes _DFR_ and _ANS_ along with the anthocyanin transport factor _AN9_ in the cactus _Astrophytum myriostigma_ resulted in anthocyanin pigmentation. The authors postulate that a lack of transcriptional activation by the MYB transcription factor could be the reason for the lack of anthocyanin producation in betalain-pigmented species.



### (30) Anthocyanin transport in sweetpotato ###

> **_<a href="https://twitter.com/boas_pucker/status/1441288791793094668">TWEET</a>:_** IbGSTF4 was identified as an anthocyanin transport-associated protein in sweetpotato. Gene expression correlates with anthocyanin pigmentation and the tt19 mutant was complemented #FlavonoidFriday (details: https://bit.ly/2WXA27m)

<a href="https://twitter.com/boas_pucker/status/1441288791793094668">
<img alt="Anthocyanin transport in sweetpotato (Tweet #30)" src="https://pbs.twimg.com/media/E-rcudkXEAIh04S?format=jpg&name=large" width="40%" height="40%">
</a>

[Kou et al., 2019](https://doi.org/10.1016/j.plaphy.2018.12.028) identified a glutathione S-transferase in sweetpotato (_IbGSTF4_, _Ipomoea batatas_) that is involved in the anthocyanin transport. The first evidence for the identification was high sequence similarity to the previously chracterized AN9 in petunia.
_IbGSTF4_ showed strong gene expression in purple-fleshed tissues. A complementation of the _Arabidopsis thaliana tt19_ mutant with _IbGSTF4_ was successful. _IbGSTF4_ expression was not activated by the anthocyanin regulating MYB (IbMYB1).


### (31) Plant Biotechnology and Bioinformatics group starts research on specialized metabolites at TU Braunschweig ###

> **_<a href="https://twitter.com/boas_pucker/status/1443825506021888003">TWEET</a>:_** Today, @PuckerLab starts research on specialized metabolism at @tuBraunschweig. Genomics & bioinformatics will help to unlock the secrets of plants. Details: https://lnk.tu-bs.de/nvToVH #PlantSci #FlavonoidFriday

<a href="https://twitter.com/boas_pucker/status/1443825506021888003">
<img alt="Plant Biotechnology and Bioinformatics (Tweet #31)" src="https://pbs.twimg.com/media/FAl8hVBWQAEFgVe?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

Details about the research in the group and open positions can be found on the website: https://www.tu-braunschweig.de/en/ifp/pbb


### (32) Bifunctional FLS in rapeseed ###

> **_<a href="https://twitter.com/boas_pucker/status/1446362223387971586">TWEET</a>:_** A flavonol derivative is a major off-taste component in rapeseed protein isolates preventing use in human consumption. Read about bifunctional FLS in B. napus seeds:https://bit.ly/3ieXWTO #FlavonoidFriday (details: https://bit.ly/39IxL3h)

<a href="https://twitter.com/boas_pucker/status/1446362223387971586">
<img alt="Bifunctional FLS in rapeseed (Tweet #32)" src="https://pbs.twimg.com/media/FAZXihWVQAUhCDS?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Schilbert et al., 2021](https://doi.org/10.1101/2021.06.30.450533) characterized the FLS gene family in _Brassica napus_.



### (33) New to flavonoids? ###

> **_<a href="https://twitter.com/boas_pucker/status/1448898936979312644">TWEET</a>:_** New to flavonoids? They are exciting! Diverse in structure, function, and color. Promiscutity of the biosynthesis and decorating enzymes leads to a plethora of metabolites #FlavonoidFriday (details: https://bit.ly/3AGcjH5)

<a href="https://twitter.com/boas_pucker/status/1448898936979312644">
<img alt="New to flavonoids? (Tweet #33)" src="https://pbs.twimg.com/media/FBrywCIXIAg1Z_-?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

If you are new to flavonoids, you can get an introduction into the genes involved in the biosynthesis [here](https://www.mdpi.com/2223-7747/9/9/1103/htm). The biosynthesis is always displayed in a simplified way, because the number of modifying enzymes is huge. There are glycosyltransferases, methyltransferases, acyltransferases ... that can act on various flavonoids. Promiscuity of these enzymes is a major factor that leads to a plethora of different metabolites in plants.



### (34) Chokecherry pigmentation ###

> **_<a href="https://twitter.com/boas_pucker/status/1451435653330137092">TWEET</a>:_** Red pigmentation of _Padus virginiana_ leaves is largely dependent on activity of the anthocyanidin glycosyltransferases #FlavonoidFriday (details: https://bit.ly/3B0U53b)

<a href="https://twitter.com/boas_pucker/status/1451435653330137092">
<img alt="Chokecherry pigmentation (Tweet #34)" src="https://pbs.twimg.com/media/FCQLc_7WUBIWYrU?format=jpg&name=large" width="30%" height="30%">
</a>

[Li et al., 2021](https://doi.org/10.3390/ijms221910697) identified the anthocyanin glycosyltransferases in _Padus virginiana_ (chokecherry) as the bottleneck in the coloration of leaves. Homologs of BZ1 and other flavonoid glucosyltransferases show substantially increased gene expression in red leaves compared to green leaves. This tree is an important ornamental species thus understanding the mechanism could pave the way for improvements.


### (35) Black rice pigmentation ###

> **_<a href="https://twitter.com/boas_pucker/status/1453972365885509634">TWEET</a>:_** Upregulation of the anthocyanin biosynthesis by OsC1 improves the stress tolerance of rice #FlavonoidFriday (details: https://bit.ly/2ZyBHlM)

<a href="https://twitter.com/boas_pucker/status/1453972365885509634">
<img alt="Black rice pigmentation (Tweet #35)" src="https://pbs.twimg.com/media/FCzqyflXMAQPRnI?format=jpg&name=medium" width="30%" height="30%">
</a>

[Upadhyaya et al., 2021](https://doi.org/10.1111/ppl.13583) investigated the tissue-specific regulation of anthocyanin biosynthesis in rice through the comparison of white and black accessions. _OsC1_, a gene encoding a R2R3-MYB transcription factor, was overexpressed in white rice to study its function. Developmental stages of white and black rice were compared and revealed a strong correlation between the antioxidant activity and the anthocyanin level (mostly cyanidin 3-glucoside). Overexpression of _OsC1_ resulted in elevated expression levels of the anthocyanin biosynthesis genes. The OX lines also showed a lower production of reactive oxyen species (ROS), less photosynthetic damage, and less membrane damage. Natural expression of _OsC1_ correlates with the expression of anthocyanin biosynthesis genes in black rice panicle.


### (36) Hydroxylation patterns of flavonoids ###

> **_<a href="https://twitter.com/boas_pucker/status/1456524181978456067">TWEET</a>:_** Hydroxylation patterns of anthocyanins and flavonols are well correlated in the flowers of Iochrominae, but correlation is low between flower and leaves #FlavonoidFriday (details: https://bit.ly/3o2ePDr)

<a href="https://twitter.com/boas_pucker/status/1456524181978456067">
<img alt="Hydroxylation patterns of flavonoids (Tweet #36)" src="https://pbs.twimg.com/media/FDX0_-qXEAES_0p?format=jpg&name=large" width="30%" height="30%">
</a>

[Berardi et al., 2016](http://dx.doi.org/10.1016/j.phytochem.2016.05.007) investigated the hydroxylation patterns of flavonoids in Iochrominae (Solanaceae). Anthocyanins and flavonols can have between one and three hydroxy groups which determine their biochemical properties. The color of anthocyanins depends on the hydroxylation state with trihydroxylated anthocyanins appearing as blue pigments. The most abundant anthocyanin usually determines the observed flower color, but other pigments like carotenoids can have an influence. Different Iochrominae were analyzed with respect to the abundance of differently hydroxylated flavonoids. Species with a high amount of trihydroxylated anthocyanins also showed a high amount of trihydroxylated flavonols (myricetin glcyosides). Species with a high amount of dihydroxylated anthocyanins also showed a high amount of dihydroxylated flavonols (quercetin glcyosides). Species with a high amount of monohydroxylated anthocyanins also showed a high amount of monohydroxylated flavonols (kaempferol glcyosides). Myricetin was only detected in the purple flowers probably due to the exclusive presence of F3'5'H in these flowers.



### (37) Anthocyanin colors ###

> **_<a href="https://twitter.com/boas_pucker/status/1459060897385795584">TWEET</a>:_** Anthocyanins appear in a wide range of colors depending on decorations/modification of Pelargonidin, Cyanidin, Delphinidin, Peonidin, Petunidin, Malvidin (and more) #FlavonoidFriday (details: https://bit.ly/3oiGgZO)

<a href="https://twitter.com/boas_pucker/status/1459060897385795584">
<img alt="Anthocyanin colors (Tweet #37)" src="https://github.com/bpucker/figures/blob/main/FlavonoidFriday_AnthocyaninColors.png?raw=true" width="50%" height="50%">
</a>

(original figure contained a mistake concerning the structures)

Pelargonidin, Cyanidin, and Delphinidin are the first anthocyanidins and differ in the number of hydroxy groups. The color ranges from orange over purple to blue. Additional modifications are Peonidin (O-methylated Cyanidin), Petunidin (O-methylated Delphinidin), and Malvidin (2x O-methylated Delphinidin). The addition of sugar moieties turns the anthocyanidins into anthocyanins.


### (38) Anthocyanin degradation ###

> **_<a href="https://twitter.com/boas_pucker/status/1461597612696502277">TWEET</a>:_** Potato tubers accumulate anthocyanins and maintain stable levels, while purple-fruited genotypes of eggplant and pepper show high levels in unripe fruits which decrease during ripening #FlavonoidFriday (details: https://bit.ly/3HAiHUR)

<a href="https://twitter.com/boas_pucker/status/1461597612696502277">
<img alt="Anthocyanin degradation (Tweet #38)" src="https://pbs.twimg.com/media/FEga17BWYAsw2A3?format=jpg&name=large" width="30%" height="30%">
</a>

[Liu et al., 2018](https://doi.org/10.3389/fchem.2018.00052) reviewed anthocyanin degradation mechanisms in the Solanaceous vegetables. The color of eggplant, pepper, and potato is influenced by anthocyanins. Besides the well characterized biosynthesis the degradation of anthocyanins also contributes to the observed pigment levels. There is a genetic factor that influences the speed of the anthocyanin degradation. However, the molecular mechanism of this degradation remains largely unknown. Potential degradation pathways are (1) oxidation by a peroxidase and (2) deglycosylation by a -glucosidase and oxidation by a polyphenol oxidase or peroxidase.


### (39) Banana ODDs ###

> **_<a href="https://twitter.com/boas_pucker/status/1464134326333935619">TWEET</a>:_** Flavanone 3-hydroxylase (F3H), flavonol synthase (FLS), and anthocyanidin synthase (ANS) belong to the 2-oxoglutarate-dependent dioxygenase (ODD) gene family #FlavonoidFriday (details: https://bit.ly/3lasQxZ)

<a href="https://twitter.com/boas_pucker/status/1464134326333935619">
<img alt="Banana ODDs (Tweet #39)" src="https://pbs.twimg.com/media/FFEpJjXXMA41eib?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

[Busche et al., 2021](https://doi.org/10.3389/fpls.2021.701780) characterized three members of the 2-oxoglutarate-dependent dioxygenase (ODD) family in _Musa acuminata_. First, flavanone 3-hydroxylase (F3H), flavonol synthase (FLS), and anthocyanidin synthase (ANS) functions were checked through a bioconversion assay. Next, these functions were also validated _in planta_ through complementation of the corresponding _Arabidopsis thaliana_ mutant lines. This study provides the basis for genetic engineering of banana plants towards enhanced antioxidant activity and increased nutritional value (biofortification).


### (40) Proanthocyanidins ###

> **_<a href="https://twitter.com/boas_pucker/status/1466671041892155397">TWEET</a>:_** Proanthocyanidins (PAs) comprise 2-30 subunits. Polymerization of the flavan-3-ol subunits still poses many questions #FlavonoidFriday (details: https://bit.ly/3G0et7A)

<a href="https://twitter.com/boas_pucker/status/1466671041892155397">
<img alt="Proanthocyanidins (Tweet #40)" src="https://pbs.twimg.com/media/FFoEofZX0AMEK5x?format=jpg&name=medium" width="30%" height="30%">
</a>

[Constable, 2018](https://pubs.acs.org/doi/10.1021/acs.jafc.8b02950) reviews the current knowlede about proanthocyanidins (PAs). While the biosynthesis of the flavan-3-ol precursors is well understood, the polymerization to PAs still poses questions. 2,3-cis-flavan-3-ols (epicatechin, epigallocatechin) and 2,3-trans-flavan-3-ols (catechin, gallocatechin) are PA subunits. Between 2 and 30 of these subunits are connected in the polymerization process. PAs are abundant in trees and in the coats of seeds. Living tissues contain PAs in the central vacuole. It is assumed that MATE transporteres are required to import 3-O-glycosylated precursors thus the polymerization might happen in the vacuole. The closely related species _Populus angustifolia_ and _P. fremontii_ differ substantially in their polymer size thus indicating an enzymatic basis. The hydroxylation of the B-ring can differ between precursors with procyanidin-type subunits beeing the most abundant onces in most species. The biosynthesis of prodelphinidin-type precursors requires F3'5'H activity. Further decoration of PAs with acyl groups is possible and might boost their health benefits.


### (41) Loss of rice pigments ###

> **_<a href="https://twitter.com/boas_pucker/status/1469207757211402243">TWEET</a>:_** MYB (C1), bHLH (S1), and DFR (A1) control the pigmentation of rice. Loss of purple hull color occurred independently in many cultivars #FlavonoidFriday (details: https://bit.ly/3xQnPjn)

<a href="https://twitter.com/boas_pucker/status/1469207757211402243">
<img alt="Loss of rice pigments (Tweet #41)" src="https://pbs.twimg.com/media/FCzqyflXMAQPRnI?format=jpg&name=medium" width="30%" height="30%">
</a>

[Sun et al., 2018](https://academic.oup.com/jxb/article/69/7/1485/4816143) studied the C-S-A gene system that controls the hull pigmentation in rice. _C1_ encodes a R2R3-MYB transcription factors, _S1_ encodes a bHLH protein, and _A1_ encodes the anthocyanin biosynthesis enzyme DFR. A genetic investigation of differently pigmented mutants (brown, purple, yellow) revealed that three loci are involved in the control of hull pigmentation. Epistasis was inferred from crossing experiments. The interaction between the MYB C1 and the bHLH S1 was demonstrated in yeast-two-hybrid experiments. Interestingly, anthocyanins are only produced in the purple pigmented cultivars with cyanidin 3-O-glucoside beeing one major contributor. The loss of this pigmentation occurred independently in _Oryza sativa_ and _Oryza japonica_. Different molecular events were identified within both species thus suggesting multiple independent losses. Black pigmentation is the results of a gain-of-function mutation in the bHLH.


### (42) Biochemical diversity of flavonoids ###

> **_<a href="https://twitter.com/boas_pucker/status/1471736926415863809">TWEET</a>:_** Biochemical diversity of flavonoids is based on hydroxylation, glycosylation, methylation, acylation, and many other modifications #FlavonoidFriday (details: https://bit.ly/3dKUQnN) 

<a href="https://twitter.com/boas_pucker/status/1471736926415863809">
<img alt="Biochemical diversity of flavonoids (Tweet #42)" src="https://pbs.twimg.com/media/FGgMKCDWQAI4B_8?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

Hydroxylation patterns and other modifications of the aglcyon distinguish between different types of flavonoids. Aglycons can be modified through the successive addition of sugar moieties. The 3 and 5 position of the C ring are most frequently modified. Different sugar groups and can be attached by a large set of different glycosyltransferases as recently reviewed by [Tohge et al., 2017](https://doi.org/10.1093/jxb/erx177).


### (43) Red pigmentation of poinsettia ###

> **_<a href="https://twitter.com/boas_pucker/status/1474273641340755989">TWEET</a>:_** Euphorbia pulcherima cultivar 'Harvest Orange' has a frameshift mutation in the flavonoid 3'-hydroxylase gene (non-functional enzyme) compared to Christmas Beauty and Christmas Feeling #FlavonoidFriday (details: https://bit.ly/33fCWaM)

<a href="https://twitter.com/boas_pucker/status/1474273641340755989">
<img alt="Red pigmentation of poinsettia (Tweet #43)" src="https://pbs.twimg.com/media/FGgM-kRXoAU7X2M?format=jpg&name=large" width="50%" height="50%">
</a>

[Nitarska et al., 2018](https://bmcplantbiol.biomedcentral.com/articles/10.1186/s12870-018-1424-0) investigated red hues in the bracts of different _Euphorbia pulcherima_ cultivars. Dark red coloration of the cultivars Christmas Beauty and Christmas Feeling is based on cyanidin derivatives. Other cultivars show an orange-red bract color that is caused by pelargonidin derivatives. DFRs of _E. pulcherima_ prefer dihydromyricetin and dihydroquercetin over dihydrokaempferol thus producing more cyanidin than pelargonidin. A frameshift mutation in the F3'H of 'Harvest Orange' results in a premature stop codon and an inactive enzyme. Although this cultivar is heterozygot for this mutation, only the non-functional allele is expressed in the bracts.


### (44) Red leaves of _Pennisetum setaceum_ 'Rubrum' ###

> **_<a href="https://twitter.com/boas_pucker/status/1476810355250655232">TWEET</a>:_** Gene expression changes in chlorophyll and flavonoid biosynthesis are associated with color changes in Pennisetum setaceum Rubrum upon changes of sun light exposure #FlavonoidFriday (details: https://bit.ly/3pRbBn8)

<a href="https://twitter.com/boas_pucker/status/1476810355250655232">
<img alt="Red leaves of _Pennisetum setaceum_ 'Rubrum' (Tweet #44)" src="https://pbs.twimg.com/media/FGgN_MPWYAE3GBC?format=jpg&name=medium" width="50%" height="50%">
</a>

[Zhu et al., 2020](https://doi.org/10.1371/journal.pone.0242618) investigated the genes responsible for the redish pigmentation of the ornamental grass _Pennisetum setaceum_ 'Rubrum'. The redish pigmentation is induced by light exposure.  An RNA-seq experiment was performed to find differences in genes expression between leaves in the dark and those exposed to strong sun light. T0 are leaves under normal condition. These leaves turn green if shading is provided for 12 days (T1). The leaves turn red again if exposed to sun light (T2, T3, T4). RNA-seq revealed that the chlorophyll biosynthesis is activated when leaves are moved from light exposure to shadow. Genes involved in the flavonoid biosynthesis showed down-regulation once the sun light is blocked off. This can explain the shift from red to green when protecting plants from the sun light. The color changes back to redish once the plants are exposed to sun light again.


### (45) KIPEs ###

> **_<a href="https://twitter.com/boas_pucker/status/1479347070855270401">TWEET</a>:_** Starting 2022 with research on flavonoids? KIPEs can help you to find the flavonoid biosynthesis genes in your plant species of interest: https://lnk.tu-bs.de/SYzXMe #FlavonoidFriday (details: https://bit.ly/3pRbBn8)

<a href="https://twitter.com/boas_pucker/status/1479347070855270401">
<img alt="KIPEs (Tweet #45)" src="https://pbs.twimg.com/media/FGgOhLZXIAUJqTh?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

(Figure source: doi:[10.3390/plants9091103](https://www.mdpi.com/2223-7747/9/9/1103/htm))

[KIPEs](https://github.com/bpucker/KIPEs) can be applied to identify the genes of the flavonoid biosynthesis in a species with a new transcriptome/genome sequence assembly. This can be helpful to understand stress response or pigmentation patterns. Details are described in the corresponding publication ([Pucker et al., 2020](https://www.mdpi.com/2223-7747/9/9/1103/htm)).



### (46) Tomato fruit colours ###

> **_<a href="https://twitter.com/boas_pucker/status/1481947954718126081">TWEET</a>:_** Engineered anthocyanin biosynthesis allows modulation of tomato fruit colours ranging from 'Crimson' over 'Magenta' to 'Indigo' #FlavonoidFriday (details: https://lnk.tu-bs.de/Ro3vE7)

<a href="https://twitter.com/boas_pucker/status/1481947954718126081">
<img alt="Tomato fruit colours (Tweet #46)" src="https://pbs.twimg.com/media/FJDmK64XsAU4u3J?format=jpg&name=large" width="50%" height="50%">
</a>

[Butelli et al., 2021](https://doi.org/10.3390/horticulturae7090327) genetically modified tomatoes to achieve different colouration of the fruits. Naturally, the fruit is not coloured by anthocyanins, but by carotenoids. The heterologous expression of the three transcription factors _AmDel_ (bHLH), _AmRos1_ (MYB), and _AtMYB12_ together with _AmDFR_ and combined with a f3'5'h mutation (premature stop codon) allowed to span a huge range of fruit colours. The constructed lines differ in the major anthocyanin that is produced: pelanin in 'Crimson', peonanin in 'Magenta', and petanin in 'Indigo'. 



### (47) Anthocyanins in carrots ###

> **_<a href="https://twitter.com/boas_pucker/status/1484484669533949956">TWEET</a>:_** Carrots are usually orange due to carotenoids, but high levels of anthocyanins can result in purple or black coloration of carrots #FlavonoidFriday (details: https://lnk.tu-bs.de/DRJ9M5)

<a href="https://twitter.com/boas_pucker/status/1484484669533949956">
<img alt="Anthocyanins in carrots (Tweet #47)" src="https://pbs.twimg.com/media/FJXUXnOX0AUpWd1?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Iorizzo et al., 2020](https://www.mdpi.com/2073-4425/11/8/906) reviewed the anthocyanin biosynthesis in carrots. Carrot extracts are used as "natural" colorants of nutritional products. Anthocyanins are approved in the USA and Europe, but products must be labeld with E163 to show that these pigments are included. Carrots are rich in anthocyanins and lack off-taste components that are preventing the use of other plant species. The wide color range of these anthocyanins is another advantage. Anthocyanins account for a major fraction of the phenolic compounds in carrots. Purple carrots have often five different types of cyanidin glycosides (2 are non-acylated and 3 are acylated). Many studies reported a dominance of the acylated anthocyanins over the non-acylated anthocyanins. Acylation with ferulic (Cy3XFGG), sinapic (Cy3XSGG), and coumaric acid (Cy3XCGG) are the most abundant cyanidin glycoside modifcations in carrot roots. These acylated anthocyanins are better suited for food coloration than non-acylated anthocyanins due to their higher stability at higher pH values. Four major quantitative trait loci (QTL) were identified that are responsible for the purple pigmentation of carrots. In total, 159 potential anthocyanin biosynthesis genes were detected in the carrot genome sequence annotation. No _F3'5'H_ was identified suggesting that it is not present in carrots. _FNS_ and _FLS_ show low expression in roots of purple carrots suggesting that substrate competition needs to be avoided for high anthocyanin formation. _DFR1_ was found to be up-regulated in all studies suggesting that this is a crucial point in controlling the anthocyanin content. DcMYB6 was identified as important anthocyanin activator in some, but not in all purple cultivars. DcMYB7 and DcMYB113 were identified as additional anthocyanin activators. These MYBs might activate the anthocyanin biosynthesis in slightly different ways resulting in different ratios of anthocyanin types. A bHLH transcription factor and a GST were identified in other QTL. The modification of anthocyanin biosynthesis genes by genome editing or the application of co-pigmentation could be future research fields.



### (48) Anthocyanin color change ###

> **_<a href="https://twitter.com/boas_pucker/status/1486957214522818561">TWEET</a>:_** Increase in the vacuolar pH of Ipomoea tricolor results in a color change from purplish red to blue. NHX1 imports Na+ into the vacuole in exchange for H+ #FlavonoidFriday (details: https://lnk.tu-bs.de/W5DyNC)

<a href="https://twitter.com/boas_pucker/status/1486957214522818561">
<img alt="Anthocyanin color change (Tweet #48)" src="https://pbs.twimg.com/media/FKF-MCWWUAM1LcJ?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Yoshida et al., 2005](https://academic.oup.com/pcp/article/46/3/407/1817366) observed that a shift in the vacuolar pH of _Ipomoea tricolor_ (morning glory) from 6.6 to 7.7 results in a color change from purplish red to blue. Generally, a shift towards alkaline pH in anthocyanin-pigmented plants leads to blue coloration while a shift towards acidic pH shifts the color back to red. A Na+/H+ exchanger (NHX1) was identified in the tonoplasts of _I. tricolor_ petals. This transporter is most likely responsible for the pH shift. Anthocyanins would loose their color under alkaline conditions unless they are polyacylated and stabalized by intramolecular stacking of caffeoyl residues to the chromophore. Expression of _NHX1_ coincided with anthocyanin pigmentation in the petal cells. This correlation was strong in a time course experiment around the flower opening suggesting that pH change is synchronized with flower opening.



### (49) Glycosylation and acylation of anthocyanins ###

> **_<a href="https://twitter.com/boas_pucker/status/1489524135369662468">TWEET</a>:_** Modification of anthocyanins is based on glycosylation and acylation catalyzed by enzymes of different families. Some are located in the cytoplasm and some in the vacuole #FlavonoidFriday (details: https://lnk.tu-bs.de/ZYq1PT)

<a href="https://twitter.com/boas_pucker/status/1489524135369662468">
<img alt="Glycosylation and acylation of anthocyanins (Tweet #49)" src="https://pbs.twimg.com/media/FKvT45UWYAESJUf?format=jpg&name=large" width="50%" height="50%">
</a>

[Sasaki et al., 2014](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6271837/) reviewed the enzymes involved in the glycosylation and acylation of anthocyanins: glycosyltransferases (GTs) and acyltransferases (ATs). Various decorations of anthocyanins influence their biochemical properties e.g. their stability. Cytoplasmic GTs use UDP-sugar as donor while vacuolar GTs use aromatic acyl-glucose or p-hydroxybenzoylglucose. Usually the 3-position and the 5-position of anthocyanins are modified by the addition of sugars. There are species-specific differences. Vacuolar GTs belong to the glycoside hydrolase family 1 (GH1) which was initially discovered to hydrolyze glycosides. These acyl-glucose dependent anthocyanin glucosyltransferases (AAGTs) contain a signal peptide that determines their vacuolar localization.
Cytoplasmic ATs use acyl-CoA as donor while vacuolar ATs use malylglucose and sinapoylglucose. Acylation can take place on sugar moities in the 5-position and 3'-position. This happens while only one of these positions is glycosylated. Cytoplasmic ATs belong to the BAHD family that was named after the first identified members and their functions. Members of the serine carboxypeptidase-like (SCPL) are ATs located in the vacuole. 


### (50) Flavonoid metabolons ###

> **_<a href="https://twitter.com/boas_pucker/status/1492038193406828545">TWEET</a>:_** Flavonoid biosynthesis enzymes cluster in metabolons around the membrane-bound cytochrome P450s. Metabolons result in efficient substrate channeling #FlavonoidFriday (details: https://lnk.tu-bs.de/gMfc5N)

<a href="https://twitter.com/boas_pucker/status/1492038193406828545">
<img alt="Flavonoid metabolons (Tweet #50)" src="https://pbs.twimg.com/media/FLGCkdaXIAkb4vm?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Nakayama et al., 2019](https://doi.org/10.3389/fpls.2019.00821) reviewed the role of metabolons (clusters of enzymes participating in the same pathway) in the flavonoid biosynthesis. Flavonoids show numerous lineage-specific modifications that result in an enormous number of thousands of different metabolites. This chemodiversity of flavonoids forms the basis of specific traits that provide an evolutionary benefit to these plants. The cytochrome P450 monooxygenases (FNS II, F3'H, F3'5'H, IFS) are assumed to be bound to the ER membrane, while the other enzymes of the flavonoid biosynthesis are most likely soluble. Flavonoid metabolons were first postulated by Stafford in 1974. Evidence for their existence in multiple species was provided in the following years. The interaction of enzymes in the flavonoid biosynthesis was demonstrated via Y2H and other interaction experiments in multiple species. Flavonoid metabolons are probably not linear, but a globular construct of multiple enzymes. Non-enzymatic proteins like CHI-like or non-functional paralogs can be part of these metabolons. Different paralogs can show substantial differences in there affinity to other enzymes of the flavonoid biosynthesis. Interaction of FNS II with DFR and ANS suggest a scaffolding role, because FNS II is not required for the anthocyanin biosynthesis. Loss of FNS II leads to a reduced formation of anthocyanins. 


### (51) F3H evolved from FNS I ###

> **_<a href="https://twitter.com/boas_pucker/status/1494582460327288835">TWEET</a>:_** F3H of seed plants most likely evolved from FNS I. While FNS I is still widely distributed across liverworts, it was lost in most seed plant lineages #FlavonoidFriday (details: https://lnk.tu-bs.de/MjK024)

<a href="https://twitter.com/boas_pucker/status/1494582460327288835">
<img alt="F3H evolved from FNS I (Tweet #51)" src="https://pbs.twimg.com/media/FLvvgwKXEAY_4LX?format=jpg&name=large" width="50%" height="50%">
</a>

[Li et al., 2020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7723094/pdf/PP_202001185DR1.pdf) investigated the distribution and function of flavone synthase I (FNS I) candidates in the liverworts. Their investigation suggest that the flavone biosynthesis is an old pathway that even predates flavonol biosynthesis. F3H of seed plants seems to have emerged from a FNS I lineage. FNS I undetectable in most seed plant species. The presence of FNS II in seed plants could explain the loss of FNS I in most of them. FNS I and II are funtionally redundant, but belong to different gene families.


### (52) OvPAP2 activates anthocyanin biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1497104076563267614">TWEET</a>:_** Ectopic expression of _Orychophragmus violaceus_ _PAP2_ in rapeseed shifts the flower color from yellow to red(ish) #FlavonoidFriday (details: https://lnk.tu-bs.de/O8hQeG)

<a href="https://twitter.com/boas_pucker/status/1497104076563267614">
<img alt="OvPAP2 activates anthocyanin biosynthesis (Tweet #52)" src="https://pbs.twimg.com/media/FMYr0cHWUAgJKTq?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Fu et al., 2017](https://doi.org/10.1111/pbi.12777) investigated the molecular basis of flower pigmentation in _Orychophragmus violaceus_ through the comparison of colorless mutants against the anthocyanin-pigmented wild type. The mutant was characterized by a lack of gene expression of the _DFR_, _ANS_, and _TT19_ homologs. The bHLH _GL3_ (Bra025508) revealed no expression in the white varieties which could be an explanation for the observed phenotype. For unknown reasons, the authors continued to investigate the extopic expression of _O. violaceus_ _PAP2_ in rapeseed. One pair of _O. violaceus_ chromosomes was introduced into the closely related rapeseed. Overexpression of _OvPAP2_ in _Arabidopsis thaliana_ resulted in an enhanced anthocyanin accumulation in various plant parts including the flowers. This overexpression also caused a stronger expression of several anthocyanin biosynthesis genes. Changing the flower color of rapeseed from yellow to red would be a major step in pest control.



### (53) Apiaceae FNS evolution ###

> **_<a href="https://twitter.com/boas_pucker/status/1499648338106494976">TWEET</a>:_** Apiaceae harbour an individual FNS I lineage. This FNS I evolved through tandem gene duplication from F3H #FlavonoidFriday (details: https://lnk.tu-bs.de/Quopfb & https://doi.org/10.1101/2022.02.16.480750)

<a href="https://twitter.com/boas_pucker/status/1499648338106494976">
<img alt="Apiaceae FNS evolution (Tweet #53)" src="https://pbs.twimg.com/media/FM848TbWUAAbezd?format=jpg&name=medium" width="50%" height="50%">
</a>

[Pucker & Iorizzo, 2022](https://doi.org/10.1101/2022.02.16.480750) described the evolution of _FNS I_ in the Apiaceae. _FNS I_ evolved through tandem duplication of _F3H_ followed by neo-functionalization. Only a few amino acid substitutions are required to shift the activity of a protein from F3H to FNS. This emergence of _FNS I_ in the Apiaceae is striking, because flavone biosynthesis is catalyzed by FNS II in most seed plants. It is even assumed that _F3H_ evolved from an _FNS I_ lineage in the early land plants: from FNS I to F3H and back again.



### (54) Hyperacidification of vacuoles leads to colour change ###

> **_<a href="https://twitter.com/boas_pucker/status/1502185053740101632">TWEET</a>:_** Petunia P-ATPases PH1 and PH5 cause hyperacidification of vacuoles. Vacuolar pH determines the colour of anthocyanins. Low vacuolar pH in petunia results in a blue colouration #FlavonoidFriday (details: https://lnk.tu-bs.de/kSd68J)

<a href="https://twitter.com/boas_pucker/status/1502185053740101632">
<img alt="Hyperacidification of vacuoles leads to colour change (Tweet #54)" src="https://pbs.twimg.com/media/FNjHV8GVgAo5INS?format=jpg&name=large" width="50%" height="50%">
</a>

[Faracao et al., 2014](http://dx.doi.org/10.1016/j.celrep.2013.12.009) investigated the involvement of two P-ATPases in the hyperacidification of the petunia vacuoles in petals that determines the flower colour. Vacuoles are characterized by a midly acidic pH that can be maintained by V-ATPases and pyrophosphatases. The pH of the vacuole determines the colour of anthocyanins that are imported into this vacuole. Enzyme activities and transport processes are also dependent on the pH. Petunia petals are characterized by a hyperacidification of their vacuoles that requires additional proton transporters. If this hyperacidification fails, the petals turn blue. The petunia _ph5_ mutant shows such a blue colouration. _PH5_ was identified to encode a 3A family P-ATPase. While P-ATPase are usually located in the plasmamembrane, PH5 sits in the tonoplast and acidifies the vacuole. PH1 is another P-ATPase located in the tonoplast and the combined expression of _PH1_ and _PH5_ is sufficient to complement _ph_ mutants. In contrast to _ph5_, the _ph1_ mutant does not abolish proanthocyanidin accumulation in the seed. It was observed that _PH1_ expression is activated by a protein complex: AN1(bHLH)-PH4(MYB)-PH3(WRKY)-AN11(WD40). PH1 and PH5 belong to different subfamilies of P-ATPases. PH1 is unable to translocate protons across the tonoplast. The authors speculate that it might be required for Mg2+ transport to compensate the electric potential generated through the proton transport of PH5, but experiments to test this hypothesis do not provide any support. A direct interaction of PH1 and PH5 was observed and results in an enhanced activity of PH5. P-ATPases can pump protons against a larger electrochemical gradient than V-ATPases which might explain the presence of PH1 and PH5 in the tonoplast and their involvement in hyperacidification.


### (55) Connecting core and specialized metabolism ###

> **_<a href="https://twitter.com/boas_pucker/status/1504721767599857686">TWEET</a>:_** The activation of the flavonoid biosynthesis (especially anthocyanin biosynthesis) in response to high light is regulated by triosephosphate export from the chloroplasts #FlavonoidFriday (details: https://lnk.tu-bs.de/4tPfoz)

<a href="https://twitter.com/boas_pucker/status/1504721767599857686">
<img alt="Connecting core and specialized metabolism (Tweet #55)" src="https://pbs.twimg.com/media/FOC3yCEWQAcmv9b?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Zirngibl et al., 2022](https://doi.org/10.1101/2022.03.09.483619) investigated the regulatory role of triosephosphatexports from the chloroplast on the flavonoid biosynthesis. The activation of photoprotective flavonoids in response to high light conditions is not completely understood yet. Increased availability of sugars upon high light treatment provides the basis for flavonoid production. The triosephosphat transporter TPT is essential for the transport of photosynthesis products across the chloroplast membrane. Phytohormones and ROS have only a minor influence on the anthocyanin biosynthesis activation upon high light. Inactivation of the SNF1-RELATED PROTEIN KINASE 1 (SnRK1) was a prerequirement for the activation of the flavonoid biosynthesis upon high light exposure. SnRK1 is repressed by regulatory sugar-phosphates.


### (56) Up-regulation of _MdACO1_ promotes anthocyanin formation in apple ###

> **_<a href="https://twitter.com/boas_pucker/status/1507258483141271553">TWEET</a>:_** High light promotes ethylene production in apple fruits through activation of MdACO1 by the long non-coding RNA MdLNC610. This activation of the ethylene biosynthesis leads to an increased anthocyanin formation #FlavonoidFriday (details: https://lnk.tu-bs.de/bnx7D3)

<a href="https://twitter.com/boas_pucker/status/1507258483141271553">
<img alt="Up-regulation of _MdACO1_ promotes anthocyanin formation in apple (Tweet #56)" src="https://pbs.twimg.com/media/FOq4amaXsCQ9Rr2?format=jpg&name=medium" width="50%" height="50%">
</a>

[Yu et al., 2022](https://doi.org/10.1093/plphys/kiac049) investigated the effects of high light on the ripening of apple fruits. They noticed an increased ethylene and anthocyanin formation under high light treatment compared to moderate light condition. Expression of the ethylene and anthcoyanin biosynthesis genes was increased under high light. Inhibition of the ethylene biosynthesis also resulted in a blocked anthocyanin biosynthesis suggesting that the anthocyanin biosynthesis dependes on the ethylene biosynthesis. The gene encoding the long non-coding RNA MdLNC610 (MSTRG.97610.1) is located downstream of the ethylene biosynthesis gen _MdAC01_. Both genes show strong co-expression. While the _MdACO1_ promoter does not have any light responsive cis-elements, the _MdLNC610_ promoter does have a light responsive element. Therefore, this long non-coding RNA might be the regulator of the ethylene biosynthesis. An up-regulation of _ACO1_ resulted in an increased anthocyanin formation in apple fruits.


### (57) Anthocyanic Vacuolar Inclusions ###

> **_<a href="https://twitter.com/boas_pucker/status/1509780100492914690">TWEET</a>:_** Anthocyanic vacuolar inclusions (AVIs) are clusters of anthocyanins in the vacuole. Aromatic acylation of cyanidin 3-O-glucoside can be used to induce the formation of AVIs. pH can influence the AVI formation #FlavonoidFriday (details: https://lnk.tu-bs.de/jPGkYS)

<a href="https://twitter.com/boas_pucker/status/1509780100492914690">
<img alt="Anthocyanic Vacuolar Inclusions (Tweet #57)" src="https://pbs.twimg.com/media/FPPDE19XsCwaapJ?format=jpg&name=large" width="50%" height="50%">
</a>

[Kallam et al, 2017](https://www.cell.com/current-biology/pdfExtended/S0960-9822(17)30164-1) investigated the formation of anthocyanic vaucolar inclusions (AVIs) in tobacco. AVIs are clusters of anthocyanins in the vacuole which are not enclosed by a membrane. Heterologous expression of different acyltransferases caused the formation of AVIs in tobacco where no AVIs are formed naturally. This seems to be due to aromatic acylations. Aliphatic acylation did not induce the formation of AVIs. Precipitation studies with different pH values and salt concentrations identified a mechanism that could explain the formation of AVIs composing anthocyanins with aromatic acyl moieties. AVI formation _in planta_ only occurs if the anthocyanin formation and accumulation lead to high concentrations. AVI formation is pH-dependent and can be suppressed by acidification or activation by alkalinization.
Decoration of anthocyanins with aromatic acyl groups is one way to achieve blue flowers. 5-O-glycosylation might be a mechanism to maintain a high solubility and this mechanism could have been lost in species without acyltransferase activity. Overexpression of a _5GT_ reduces the AVI formation. 


### (58) Peach flower color ###

> **_<a href="https://twitter.com/boas_pucker/status/1512316813077463044">TWEET</a>:_** Pigmentation differences between white, pink, and red peach flowers are due to an InDel in the anthocyanin transporter gene Riant #FlavonoidFriday (details: https://lnk.tu-bs.de/mHvYBn)

<a href="https://twitter.com/boas_pucker/status/1512316813077463044">
<img alt="Peach flower color (Tweet #58)" src="https://pbs.twimg.com/media/FPzFQG8X0AQdWB7?format=jpg&name=large" width="50%" height="50%">
</a>

[Cheng et al., 2015](https://academic.oup.com/jxb/article/66/22/7227/2893283) investigated the molecular mechanisms underlying the flower color differences in peach. There are white, pink, and red flowers on the same peach tree. A small InDel in the _Riant_ gene (Regulator involved in anthocyanin transport) that encodes an anthocyanin transport protein was identified as the causal mutation. A 2bp deletion in the last exon results in a frameshift leading to a premature stop codon. White flowers carry a homozygous knock-out, while pink flowers are heterozygous.


### (59) Anthocyanin inclusions ###

> **_<a href="https://twitter.com/boas_pucker/status/1514853528942039042">TWEET</a>:_** Light can induce the darkening of anthocyanin pigmentation in maize through spread of anthocyanins from vacuolar inclusions into the vacuolar sap #FlavonoidFriday (details: )

<a href="https://twitter.com/boas_pucker/status/1514853528942039042">
<img alt="Anthocyanin inclusions (Tweet #59)" src="https://pbs.twimg.com/media/FQReHE_WUAICrCM?format=jpg&name=large" width="50%" height="50%">
</a>

[Irani & Grotewold, 2005](https://bmcplantbiol.biomedcentral.com/articles/10.1186/1471-2229-5-7) studied the influence of light on the formation and destabalization of anthocyanic vacuolar inclusion (AVI)-like structures. Overexpression of _R_ and _C1_ was sufficient to enhance the accumulation of anthocyanins independent of the light treatment. Exposure of the resulting maize cells to light caused the spread of anthocyanins from vacuolar inclusions to the whole vacuole. This caused a darker pigmentation of the maize cells.


### (60) Prenyltransferase in _Lupinus albus_ ###

> **_<a href="https://github.com/bpucker/FlavonoidFriday">TWEET</a>:_** Lupinus albus prenyltransferase LaPT2 modifies flavonol aglycons while LaPT1 acts upon isoflavonoids. Flavonols with a low degree of hydroxylation (kaempferol) are preferred substrates of LaPT2 #FlavonoidFriday (details: https://lnk.tu-bs.de/nWGc1D)

<a href="https://twitter.com/boas_pucker/status/1517390242378133506">
<img alt="Prenyltransferase in _Lupinus albus_ (Tweet #60)" src="https://pbs.twimg.com/media/FQ7MU7pWUAEPeh2?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Liu et al., 2021](https://doi.org/10.3389/fpls.2021.673337) studied the _LaPT2_ gene in white lupin (_Lupinus albus_) which encodes a prenyltransferase that acts upon the flavonols kaempferol, quercetin, and myricetin. Prenylation of flavonoids is a modification that has been reported in Leguminosae, Moraceae, Umbelliferae, and Euphorbiaceae. Health promoting properties were assigned to prenylated flavonoids thus increasing the interest to study prenyltransferases. In a previous study, LaPT1 was identified to prenylate only genistein (isoflavonoid).
_LaPT2_ is expressed in roots and shows co-expression with _LaF3H1_ and _LaFLS2_. These genes already indicate an involvement in the flavonol biosynthesis. Enzyme assays revealed that flavonols are the preferred substrate of LaPT2. This is in contrast to LaPT1 which modifies isoflavonoids. Dimethylallyl pyrophosphate (DMAPP) serves as the preferred prenyl donor for the reaction catalyzed by LaPT2. LaPT2 only accepts the flavonol aglycon which suggests that the prenylation preceds the glycosylation. LaPT2 prefers flavonols with a lower degree of hydroxylation, because conversion rate decreases from kaempferol over quercetin to mycricetin.


### (61) KIPEs3 online ###

> **_<a href="https://twitter.com/boas_pucker/status/1519926958196363264">TWEET</a>:_** Automatic annotation of flavonoid biosynthesis genes with KIPEs3 is now available as an online service: http://pbb.bot.nat.tu-bs.de/KIPEs/. Just submit a FASTA file of your species of interest to start the analysis. #FlavonoidFriday (https://lnk.tu-bs.de/R9pO5i & https://doi.org/10.3390/plants9091103)

<a href="https://twitter.com/boas_pucker/status/1519926958196363264">
<img alt="KIPEs3 online (Tweet #61)" src="https://pbs.twimg.com/media/FRemcC5XEAA1YZN?format=jpg&name=large" width="50%" height="50%">
</a>

An automatic flavonoid annotation workflow ([Pucker et al., 2020](https://www.mdpi.com/2223-7747/9/9/1103/htm)), is now available as an online service: [KIPEs3 online](http://pbb.bot.nat.tu-bs.de/KIPEs/). Users can submit a FASTA file with peptide sequences of their species of interest without having to install the tool locally.


### (62) Independent evolution of prenyltransferases ###

> **_<a href="https://twitter.com/boas_pucker/status/1522463674308341761">TWEET</a>:_** Discovery of prenyltransferases in Morus alba and Cudrania tricuspidata suggested independent evolution of this enzymatic function in different plant lineages #FlavonoidFriday (details: https://lnk.tu-bs.de/kABO2d)

<a href="https://twitter.com/boas_pucker/status/1522463674308341761">
<img alt="Independent evolution of prenyltransferases (Tweet #62)" src="https://pbs.twimg.com/media/FSAPf4VWYAACFC6?format=jpg&name=large" width="50%" height="50%">
</a>

[Wang et al., 2014](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4276850/) identified flavonoid prenyltransferases (FPTs) outside the Leguminosae. _Morus alba_ and _Cudrania tricuspidata_ harbour isoliquiritigenin 3'-dimethylallyltransferases (IDTs). Both sequences show low similarity to FPTs of the legumes and form a distant clade in a phylogentic tree. This suggest an independent evolution. The IDTs acted upon chalcones, isoflavones, and flavones. 


### (63) Purple pigmentation of tomato fruits ###

> **_<a href="https://twitter.com/boas_pucker/status/1525000391041069056">TWEET</a>:_** Novel alleles of anthocyanin biosynthesis genes form the basis for purple pigmentation of Solanum galapagense-derived tomato fruits #FlavonoidFriday (details: https://lnk.tu-bs.de/H1mzCj)

<a href="https://twitter.com/boas_pucker/status/1525000391041069056">
<img alt="Purple pigmentation of tomato fruits (Tweet #63)" src="https://pbs.twimg.com/media/FSnB6paXoAc0kfY?format=jpg&name=large" width="50%" height="50%">
</a>

[Fenstemaker et al., 2022](https://doi.org/10.1002/pld3.394) identified quantitative trait loci of the anthocyanin pigmentation in _Solanum galapagense_-derived tomato fruits. Purple pigmentation was observed in _Solanum lycopersicoides_ LA2951 and _Solanum lycopersicum_ LA1996, but not in most of the other tomato cultivars. Most cultivars have green or red fruits. _Solanum galapagense_ LA1141 shows purple pigmentation that seems to have evolved independently. Malvidin and petunidin are responsible for this pigmentation. The MYB transcription factor _atv_ seems to be non-functional in _Solanum galapagense_ LA1141, but the MYB transcription factor _AFT_ could be responsible for the purple pigmentation. A phylogenetic analysis suggest an independent evolution of this MYB towards anthocyanin regulation in _Solanum galapagense_ LA1141, because closely related cultivars show green or red pigmentation. An alternative explanation would be introgression from a red pigmented relative, because this clade shares a gain-of-function mutation in _AFT_.



### (64) Pigmentation of sweet basil ###

> **_<a href="https://twitter.com/boas_pucker/status/1527537104045105152">TWEET</a>:_** Mutations in two ANS genes of Ocimum basilicum are responsible for the lack of anthocyanins observed in green sweet basil plants with white flowers #FlavonoidFriday (details: https://lnk.tu-bs.de/4HcU0X)

<a href="https://twitter.com/boas_pucker/status/1527537104045105152">
<img alt="Pigmentation of sweet basil (Tweet #64)" src="https://pbs.twimg.com/media/FTG_HCBWYAA1slw?format=jpg&name=large" width="50%" height="50%">
</a>

[Gonda et al., 2022](https://doi.org/10.1101/2022.05.16.492104) investigated the presence/absence of anthocyanins in different sweet basil (_Ocimum basilicum_) cultivars. Some cultivars have white flowers and green leaves, while others have purple flowers and purplish leaves. The F2 of crossing a green/white with a purple plant showed a 3:1 ratio of purple to green plants suggesting a single responsible locus. The plants showed a difference in the pigmentation intensity with 50% showing light pigmentation. Association of this color phenotype with SNPs revealed a locus that explains 62% of the phenotypic variation. A candidate gene approach revealed that ANS is the only anthocyanin biosynthesis gene located in this region of the genome. Due to the ploidy of _O. basilicum_ there are two homeologous regions that harbour one ANS each: ObANS1 and ObANS2. Expression analyses did not reveal a difference between the cultivars. ObANS1 has a 1bp deletion that causes a frameshift. ObANS2 has a substitution that causes a premature stop codon. There are additional mutations that render both genes non-functional in the green/white cultivar. The presence of two non-functional ANS in the same cultivar is probably the result of breeding for anthocyanin free plants. The analysis of other cultivars suggests that the regulation of anthocyanin biosynthesis might be different between leaves and flowers.


### (65) CHS in safflower ###
> **_<a href="https://twitter.com/boas_pucker/status/1530073817707646982">TWEET</a>:_** CHS4 of safflower (Carthamus tinctorius) appears to be regulated by differential alternative splicing which influences the completeness of the resulting protein #FlavonoidFriday (details: https://lnk.tu-bs.de/HQZf3Y)

<a href="https://twitter.com/boas_pucker/status/1530073817707646982">
<img alt="CHS in safflower (Tweet #65)" src="https://pbs.twimg.com/media/FTtlr21WQAAZ94V?format=jpg&name=small" width="50%" height="50%">
</a>

[Wu et al., 2021](https://doi.org/10.1111/pbi.13586) investigated the flavonoid biosynthesis genes in the safflower (_Carthamus tinctorius_) genome. Seven CHS candidate genes were identified. _CarCHS1_, _CarCHS4_, _CarCHS5_, _CarCHS6_, and _CarCHS7_ are expressed in the flower, while _CarCHS3_ is expressed in the cotyledons. Several CHS genes showed colienearily with CHS genes in lettuce and sunflower. _CarCHS5_ and _CarCHS6_ are the result of a tandem duplication of _CarCHS4_. Genes of the flavonoid biosynthesis appear regulated at the transcriptional level and even through differential alternative splicing.


### (66) Glycosylation of flavonoids ###
> **_<a href="https://twitter.com/boas_pucker/status/1532610533521707010">TWEET</a>:_** Deglycosylation of flavonoids is used in food industry to improve taste through removal of off-taste components e.g. debittering of citrus juice #FlavonoidFriday (details: https://lnk.tu-bs.de/CZKYJg)

<a href="https://twitter.com/boas_pucker/status/1532610533521707010">
<img alt="Glycosylation of flavonoids (Tweet #66)" src="https://pbs.twimg.com/media/FUR4v4FX0AE1WQT?format=jpg&name=large" width="50%" height="50%">
</a>

[Slamova et al., 2018](10.3390/ijms19072126) reviewed the current knowledge about addition and removal of sugar moieties in the flavonoid metabolism of plants. Flavonoids account for a major proportion of phenolic compounds in fruits and vegetables. Many flavonoids were reported to have a beneficial physiological effects. Natural flavonoids are often glycosylated which improves their bioavailability. Removal of sugar groups is widely used in the food industry to improve the taste by depletion of off-taste components.


### (67) Role of MtrGSTF7 in anthocyanin transport ###
> **_<a href="https://twitter.com/boas_pucker/status/1535147247851085824">TWEET</a>:_** The gstf7 mutant of Medicago truncatula shows a lack of anthocyanins, but an increased accumulation of proanthocyanidins in the leaves. MtrGSTF7 can complement the corresponding A. thaliana mutant (tt19) #FlavonoidFriday (details: https://lnk.tu-bs.de/uwJ3X0)

<a href="https://twitter.com/boas_pucker/status/1535147247851085824">
<img alt="Role of MtrGSTF7 in anthocyanin transport (Tweet #67)" src="https://pbs.twimg.com/media/FU3iDJ5WQAA-Ea8?format=jpg&name=medium" width="50%" height="50%">
</a>

[Panara et al., 2022](https://doi.org/10.3390/plants11101318) studied the function of MtrGSTF7 in _Medicago truncatula_. The _mtrgstf7_ mutant lacks anthocyanins in the leaves, but shows an increased level of proanthocyanidins. This suggests that different GSTs are responsible for anthocyanin and proanthocyanin transport, respectively. The same observation was previously reported in _Vitis vinifera_.


### (68) Highland barley cultivars are rich anthocyanin sources ###
> **_<a href="https://pbs.twimg.com/media/FVbBGiiWYAAZhyU?format=jpg&name=large">TWEET</a>:_** Colored highland barley varieties are rich in flavonoids/anthocyanins and polyphenols. Nutritional value is high due to antioxidant and hypoglycemic properties of these compounds #FlavonoidFriday (details: https://lnk.tu-bs.de/RtSHen)

<a href="https://twitter.com/boas_pucker/status/1537683963438944256">
<img alt="Highland barley cultivars are rich anthocyanin sources (Tweet #68)" src="https://pbs.twimg.com/media/FVbBGiiWYAAZhyU?format=jpg&name=large" width="50%" height="50%">
</a>

[Jin et al., 2022](https://doi.org/10.3390/molecules27113411) investigated the polyphenol and anthocyanin composition of differently colored highland barley varieties. The color correlated with particular levels of certain anthocyanins and other flavonoids. Purple barley displayed the highest levels of anthocyanins. Black and blue bareley varieties were characterized by high levels of chalcones and flavones, while yellow and purple showed high amounts of flavonols. Cornflower pigment-3-glucoside was the dominant anthocyanin in blue, yellow and black barley varieties. In contrast, delphinidin dominated in the black barley variety. While all colored barley varieties have nutritional value due to their antioxidant and hypoglycemic value, purple and black varieties are outstanding.


### (69) Branches of the flavonoid biosynthesis ###
> **_<a href="https://twitter.com/boas_pucker/status/1540220678896586753">TWEET</a>:_** The flavonoid biosynthesis comprises multiple branches leading to flavones, aurones, isoflavones, flavonols, phlobaphenes, proanthocyanidins, and anthocyanins #FlavonoidFriday (details: https://lnk.tu-bs.de/NicYyp)

<a href="https://twitter.com/boas_pucker/status/1540220678896586753">
<img alt="Branches of the flavonoid biosynthesis (Tweet #69)" src="https://pbs.twimg.com/media/FV_OgqlWAAElI-O?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Liu et al., 2021](https://doi.org/10.3390/ijms222312824) reviewed the latest discoveries concerning the flavonoid biosynthesis. Eight branches leading to stilbenes, aurones, flavones, isoflavones, flavonols, phlobaphenes, proanthocyanidins, and anthocyanins are covered. Important intermediates are chalcones, flavanones, dihydroflavonols, and leucoanthocyanidins. MYB-bHLH-WD40 and NF-Y transcription factor complexes are involved in the transcriptional regulation of the flavonoid biosynthesis.



### (70) Anthocyanin accumulation in cotton ###
> **_<a href="https://github.com/bpucker/FlavonoidFriday">TWEET</a>:_** Red and blue light induce the accumulation of anthocyanins in cotton. The anthocyanin regulator (PAP) and anthocyanin transporter (GST) are activated under these light conditions #FlavonoidFriday (details: https://lnk.tu-bs.de/9ZCRm1)

<a href="https://twitter.com/boas_pucker/status/1542757395134156800">
<img alt="Anthocyanin accumulation in cotton (Tweet #70)" src="https://pbs.twimg.com/media/FWjZrWWXEAEd6Dm?format=jpg&name=large" width="50%" height="50%">
</a>

[Shao et al., 2022](https://doi.org/10.3389/fpls.2022.788828) investigated the impact of red and blue light on the anthocyanin accumulation of cotton (_Gossypium hirsutum_). The R2R3-MYB transcription factor _GhPAP1D_ and the anthocyanin transporter _GhGSTF12_ were identified as up-regulated under these treatment conditions. GhHY5 might be an additional regulator that controlls the flavonoid biosynthesis through GhPAP1D.

### (71) Automatic annotation of the flavonoid biosynthesis ###
> **_<a href="https://twitter.com/boas_pucker/status/1545286566238515200">TWEET</a>:_** Interested in the flavonoid biosynthesis of a new plant species? An automatic annotation of the flavonoid biosynthesis genes is enabled by this web server: http://pbb.bot.nat.tu-bs.de/KIPEs/ #FlavonoidFriday (details: https://lnk.tu-bs.de/8D2ahw & https://doi.org/10.1101/2022.06.30.498365)

<a href="https://twitter.com/boas_pucker/status/1545286566238515200">
<img alt="Automatic annotation of the flavonoid biosynthesis (Tweet #71)" src="https://pbs.twimg.com/media/FXHhzi1WYAEXd_6?format=jpg&name=large" width="50%" height="50%">
</a>

Knowledge-based Identification of Pathway Enzymes v3 (KIPEs3) is available on a [web server](http://pbb.bot.nat.tu-bs.de/KIPEs/). A recent preprint by [Rempel & Pucker, 2022](https://doi.org/10.1101/2022.06.30.498365) describes describes the novel functions. For example, KIPEs3 can also identify flavonoid decorating enzymes and protein involved in the flavonoid transport. Additionally, information about the carotenoid biosynthesis is added to enable an automatic annotation of this pathway in the core metabolism.


### (72) Modification of anthocyanins in blueberry ###
> **_<a href="https://twitter.com/PuckerLab/status/1573197975131561984">TWEET</a>:_** Do you like blueberries? Identification of genetic loci determining fruit quality trais in Vaccinium corymbosum: https://doi.org/10.3389/fpls.2022.964656 & https://lnk.tu-bs.de/q8WLDC #Genomics #Bioinformatics #FlavonoidFriday

<a href="https://twitter.com/PuckerLab/status/1573197975131561984">
<img alt="Modification of anthocyanins in blueberry (Tweet #72)" src="https://pbs.twimg.com/media/Fc-iF0HX0AE59Z-?format=jpg&name=large" width="50%" height="50%">
</a>

[Mengist et al., 2022](https://doi.org/10.3389/fpls.2022.964656) identified genomic regions harbouring genes that are associated with anthocyanin modification. Anthocynins are a major factor contributing to the nutritional value of blueberries. Four major QTLs were identified on chromosomes 1, 2, 4, and 8. Genes associated with acylation and glycosylation of anthocyanins (transferases) are located in these regions.



### (73) Two strawberry DFRs show substrate specificity differences ###
> **_<a href="https://twitter.com/boas_pucker/status/1634086686287097856">TWEET</a>:_** Strawberry has two DFRs with specific substrate preferences: DFR1 accepts only dihydrokaempferol, while DFR2 accepts only dihydroquercetin and dihydromyricetin. Resulting anthocyanins are orange/red or red/pink, respectively. Details: https://lnk.tu-bs.de/6pKrdf #FlavonoidFriday

<a href="https://twitter.com/boas_pucker/status/1634086686287097856">
<img alt="Two strawberry DFRs show substrate specificity differences (73)" src="https://pbs.twimg.com/media/Fq1JDvDWAAMublK?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Miosic et al., 2014](https://doi.org/10.1371/journal.pone.0112707) described that strawberry is characterized by the presence of two dihydroflavonol 4-reductases (DFRs) that differ in their substrate preferences. DFR1 accepts only dihydrokaempferol (DHK), but does not work on diyhdroquercetin (DHQ) or dihydromyricetin (DHM). These three dihydroflavonols differ in the number of hydroxyl groups at the B-ring. DHK has only one, DHQ has two, and DHM as three hydroxyl groups at the B-ring. DFR2 only accepts DHQ and DHM, but does not accept DHK. DFR1 leads to the production of pelargonin (orange/red), while DFR2 channels substrate towards cyanin (dark red). The DFR reaction is followed by anthocyanin synthase (ANS) and a glycosyltransferase (UGT) that turn a leucoanthocyanidin produced by DFR into a colorful anthocyanin. The availability of dihydroflavonols changes during strawberry development. DHQ is available in early phases of fruit development, but DHK dominates towards the end of the fruit development. Flavonoid 3'-hydroxylase (F3'H) is an important enzyme that converts DHK into DHQ. The corresponding gene shows low expression during flower development, which could explain the shift in dihydroflavonol abundances. This [simplified illustration of the flavonoid biosynthesis](https://www.mdpi.com/plants/plants-09-01103/article_deploy/html/images/plants-09-01103-g001.png) shows the connection of different reactions in this metabolic network. The substrate specificity of strawberry DFR1 is caused by alanine at position 133. A substitution with asparagin in DFR2 is important for the differences in substrate preference. However, asparagin alone cannot explain the substrate preference, because the same amino acid is present at this position in the broad substrate DFR from gerbera.


### (74) Chickpea seed pigmentation ###
> **_<a href="https://twitter.com/boas_pucker/status/1636630950384750592">TWEET</a>:_** Tissue-specific silencing of the transcription factor CaTT8 or the transporter CaMATE1 in chickpea type desi prevents the formation and accumulation of anthocyanins and proanthocyanidins leading to lighter seed color #FlavonoidFriday details: https://lnk.tu-bs.de/xpzZDX.

<a href="https://twitter.com/boas_pucker/status/1636630950384750592">
<img alt="Chickpea seed pigmentation (74)" src="https://pbs.twimg.com/media/Fq7cdlMWAAAU4-5?format=jpg&name=medium" width="50%" height="50%">
</a>

[Pal et al., 2022](https://doi.org/10.1101/2022.08.22.504750) investigated the seed coloration of chickpea. There are two types of chickpea that difference in the seed and flower color. Desi is characterized by dark seeds and purple flower, while kabuli shows lighter seed color and white flowers. These phenotypes suggest that the flavonoid biosynthesis is implicated. Anthocyanins and proanthocyanidins (PAs) are known to cause dark pigmentation in plants. Tissue-specific silencing of the anthocyanin/PA regulating bHLH CaTT8 or the transporter CaMATE1 in desi resulted in a phenotype that resembles kabuli. A simlified illustration of the [flavonoid biosynthesis](https://www.mdpi.com/plants/plants-11-00963/article_deploy/html/images/plants-11-00963-g001.png) and the associated [flavonoid transport](https://www.mdpi.com/plants/plants-11-00963/article_deploy/html/images/plants-11-00963-g002.png) illustrate the role of MATE in transporting anthocyanins and PAs into the vacuole for long term storage. The substitution of a single amino acid in MATE1 (A117D) was sufficient to substantially reduce the transport function. It seems that an effective block of the anthocyanin and PA biosynthesis could be achieved through lack of gene expression or lack of anthocyanin transport.

### (75) Flavonoid genetics in Vaccinium ###
> **_<a href="">TWEET</a>:_** Comprehensive review about flavonoid genetics in Vaccinium crops including blueberry, cranberry, bilberry, and lingonberry #FlavonoidFriday (details: https://doi.org/10.1093/plphys/kiad250 & https://lnk.tu-bs.de/24sVPK)

<a href="https://twitter.com/boas_pucker/status/1659453840020713473">
<img alt="Flavonoid genetics in Vaccinium (75)" src="https://pbs.twimg.com/media/FwW8EADXsAI9-UW?format=jpg&name=900x900" width="50%" height="50%">
</a>

The genus _Vaccinium_, which includes popular berry crops like blueberries, cranberries, bilberries, and lingonberries, is known for its potential health benefits. These benefits are often attributed to the high levels of flavonoids, particularly anthocyanins, which give the berries their vibrant red and blue colors. Due to the growing interest in these phytochemicals among consumers, breeders have focused on developing Vaccinium varieties with enhanced flavonoid content. Recent advancements in _Vaccinium_ genomics and genetics, as well as functional studies on flavonoid regulation, have provided insights into the control of flavonoid development and identified specific genes and genetic regions that can be targeted for crop improvement. This progress not only contributes to the enhancement of Vaccinium crops but also deepens our understanding of flavonoid regulation and biosynthesis in a wider range of fruit crops. This update discusses the recent advancements in understanding flavonoid regulation, using _Vaccinium_ as a case study, and emphasizes the significant improvements in genomic tools and functional analysis.




### FlavonoidFriday posts will become less frequent ###
Due to a lack of recent open access publications about the flavonoid biosynthesis, the frequency of posts will decrease. Some classic publications will be posted again. Additionally, we are happy to share high quality findings once they get published.


## Contact ##
Do you have a recent open access publication about the flavonoid biosynthesis that could be featured here? Please get in touch: [Boas Pucker (email)](mailto:b.pucker@tu-braunschweig.de?subject=[GitHub]FlavonoidFriday)


## References ##

<details>
<summary>(click to expand)</summary>
<p>
Pucker, B., Reiher, F. and Schilbert, H. M. Automatic identification of players in the flavonoid biosynthesis with application on the biomedicinal plant Croton tiglium. Plants 2020, 9, 1103. doi:[10.3390/plants9091103](https://www.mdpi.com/2223-7747/9/9/1103/htm).

Timoneda, A., Feng, T., Sheehan, H., Walker-Hale, N., Pucker, B., Lopez-Nieves, S., Guo, R., Brockington, S. (2019). The evolution of betalain biosynthesis in Caryophyllales. New Phytologist. doi:[10.1111/nph.15980](https://doi.org/10.1111/nph.15980)

Susanna Pollastri, Massimiliano Tattini, Flavonols: old compounds for old roles, Annals of Botany, Volume 108, Issue 7, November 2011, Pages 12251233, doi:[10.1093/aob/mcr234](https://doi.org/10.1093/aob/mcr234)

Landi, M. et al. "Multiple functional roles of anthocyanins in plant-environment interactions." Environmental and Experimental Botany 119 (2015): 4-17.

Ahn-Jarvis, J.H.; Parihar, A.; Doseff, A.I. Dietary Flavonoids for Immunoregulation and Cancer: Food Design for Targeting Disease. Antioxidants 2019, 8, 202. doi:[10.3390/antiox8070202](https://doi.org/10.3390/antiox8070202)

Sun, J., Lin, L.z. and Chen, P. (2012), Study of the mass spectrometric behaviors of anthocyanins in negative ionization mode and its applications for characterization of anthocyanins and nonanthocyanin polyphenols. Rapid Commun. Mass Spectrom., 26: 1123-1133. doi:[10.1002/rcm.6209](https://doi.org/10.1002/rcm.6209)
  
Pucker, B. & Selmar, D. Biochemistry and Molecular Basis of Intracellular Flavonoid Transport in Plants. Plants 2022, 11, 963. doi:[10.3390/plants11070963](https://doi.org/10.3390/plants11070963)
  
</p>
</details>

