# #FlavonoidFriday
This repository provides additional background to the #FlavonoidFriday tweets

## Introduction ##
The flavonoid biosnythesis is one of the best studied specialized metabolism pathways in plants. Flavonoids are produced by almost all land plants, but specific branches of this pathway are not present in all species. Flavonol biosynthesis, flavone biosynthesis, anthocyanin biosnythesis, and proanthocyanidin biosynthesis are some branches of the flavonoid biosynthesis. Products of these branches have different functions and the enzymes might compete for substrate when branches are active at the same time. Therefore, the gene expression of all players in this pathway is tightly regulated by a number of transcription factors (see Transcriptional Regulation). Fig. 1 provides a simplified overview of the flavonoid biosynthesis [(Pucker et al., 2020)](https://www.mdpi.com/2223-7747/9/9/1103/htm).


<a href="https://www.mdpi.com/2223-7747/9/9/1103/htm">
<img alt="Flavonoid biosynthesis pathway overview" src="https://www.mdpi.com/plants/plants-09-01103/article_deploy/html/images/plants-09-01103-g001.png" width="50%" height="50%">
</a>  


Fig. 1: Simplified overview of the flavonoid biosynthesis. (source:[(Pucker et al., 2020)](https://www.mdpi.com/2223-7747/9/9/1103/htm))


### Transcriptional Regulation ###
The sophisticated regulatory network of the flavonoid biosnythesis contains members of numerous transcription factor families. Among the important families are MYBs, bHLHs, WD40s, NACs, and WRKYs. Some MYBs, bHLHs, and WD40s form a complex which is called MBW.

### Mutual Exclusion of Anthocyanins and Betalains ###
While the flavonoid biosynthesis is present in (almost) all land plants, certain branches are not. One famemous example is the mutual exclusion of anthocyanins and betalains in the Caryophyllales. The yellow to red betalains are tyrosine-derived pigment group which replaced anthocyanins in several families of the Caryophyllales. Both pigments have never reported together in the same tissue and the same species. Please see [Timoneda *et al*., 2019](https://doi.org/10.1111/nph.15980) for a recent review of the complex pigment evolution in the Caryophyllales.


### Automatic annotation of the flavonoid biosynthesis ###
Do you have a new transcriptome and genome assembly of a plant species? If you are interested in studying the flavonoid biosynthesis, [KIPEs](https://github.com/bpucker/KIPEs) [(Pucker et al., 2020)](https://www.mdpi.com/2223-7747/9/9/1103/htm) could be helpful. KIPEs produces an annotation of the flavonoid biosynthesis players based on knowledge about the flavonoid biosynthesis in other species. A table of the involved genes and phylogenetic trees of all players in the pathways can serve as basis to investigate the flavonoid biosynthesis in a new species.


### Excellent reviews about the flavonoid biosynthesis ###
There are several excellent resources to get an overview of the flavonoid biosynthesis. This is an incomplete list:

Winkel-Shirley, 2001: Flavonoid Biosynthesis. A Colorful Model for Genetics, Biochemistry, Cell Biology, and Biotechnology. doi:[10.1104/pp.126.2.485](http://www.plantphysiol.org/content/126/2/485)

Grotewold, 2006: The Science of Flavonoids. [Book published by Springer](https://www.springer.com/de/book/9780387288215)

Tohge _et al_., 2017: Current understanding of the pathways of flavonoid biosynthesis in model and crop plants. doi:[10.1093/jxb/erx177](https://doi.org/10.1093/jxb/erx177)



## Tweets with addtional background ##

### (1) General Flavonoid Biosynthesis Overview ###
> **_<a href="https://twitter.com/boas_pucker/status/1365194891475750916">TWEET</a>:_** Flavonoids are a large group of phenylalanine-derived specialized metabolites in plants. Branches of the biosynthesis lead to flavonols, anthocyanins, proanthocyanins, flavones, and others. #FlavonoidFriday (src: https://bit.ly/2NRPlKw)

<a href="https://twitter.com/boas_pucker/status/1365194891475750916">
<img alt="Flavonoid biosynthesis pathway overview (Tweet #1)" src="https://pbs.twimg.com/media/Eu4yOCfWYAIMKxJ?format=jpg&name=4096x4096" width="20%" height="20%">
</a>


The background about this tweet is described above in the general introduction to the flavonoid biosynthesis.


### (2) Flavonol biosynthesis is an old branch of the flavonoid biosynthesis ###
> **_<a href="https://twitter.com/boas_pucker/status/1367746709913145345">TWEET</a>:_** Flavonols are an evolutionary old group of flavonoids. Their functions include UV & ROS protection, but also signaling & regulation of developmental processes. Flavonol production is triggered by biotic and abiotic factors #FlavonoidFriday (src: https://bit.ly/3q5Ggei)

<a href="https://twitter.com/boas_pucker/status/1365194891475750916">
<img alt="Flavonol biosynthesis pathway (Tweet #2)" src="https://pbs.twimg.com/media/EvVYRu-XUAY864Q?format=png&name=900x900" width="20%" height="20%">
</a>



### (3) Anthocyanins are colourful pigments ###
> **_<a href="https://twitter.com/boas_pucker/status/1370283419763105793">TWEET</a>:_** Anthocyanins (Greek: 'blue flower') are pigments with pH-dependent colours ranging from red to blue. Their functions include attraction of animals for pollination & seed dispersal, but also protection against abiotic stressors. #FlavonoidFriday (details: https://bit.ly/3uEZz1F)

<a href="https://twitter.com/boas_pucker/status/1370283419763105793">
<img alt="Anthocyanins (Tweet #3)" src="https://pbs.twimg.com/media/EvVbCduXMAEDII8?format=jpg&name=medium" width="30%" height="30%">
</a>


While anthocyanins are well known for their colour, it is likely that additional functions in stress response might be even more important (reviewed by Landi et al., 2015).

### (4) Proanthocyanidins are responsible for seed coat colouration ###
> **_<a href="https://twitter.com/boas_pucker/status/1372820142980272130">TWEET</a>:_** Proanthocyanidins (PA) are responsible for a dark appearance of seed coats. The biosynthesis of PAs is not completely understood and probably differs between species #FlavonoidFriday (details:https://bit.ly/38XTBjk)

<a href="https://twitter.com/boas_pucker/status/1372820142980272130">
<img alt="Proanthocyanidins (Tweet #4)" src="https://pbs.twimg.com/media/EwyzS7hWYAI6zQZ?format=jpg&name=medium" width="30%" height="30%">
</a>

LAR and ANR are involved in the PA biosynthesis in many species. However, A. thaliana lacks LAR, but is still able to produce proanthocyanidins which provide a dark colouration to the seed coat. This indicates that there must be a PA biosynthesis pathway which does not require LAR.

### (5) Dietary Flavonoids ###

> **_<a href="https://twitter.com/boas_pucker/status/1375356855279570947">TWEET</a>:_** Flavonoids are abundant phytochemicals in fruits and vegetables. Health and nutritional benefits were reported for several flavonoids including antioxidant and anti-inflammatory functions #FlavonoidFriday (details:https://bit.ly/3w3hksj)

<a href="https://twitter.com/boas_pucker/status/1375356855279570947">
<img alt="Dietary Flavonoids (Tweet #5)" src="https://pbs.twimg.com/media/ExYtlWkWEAEworj?format=jpg&name=4096x4096" width="80%" height="80%">
</a>


Flavonoids are not just responsible for beautiful flower phenotypes. These molecules can also be beneficial for human health. Flavonoids in the human diet and the potential for biomedical/biotechnological applications in the future were recently reviewed by [Ahn-Jarvis et al.](https://doi.org/10.3390/antiox8070202). They discus the releveance of increasing the bioavailability of flavonoids through agricultural engineering and targeted food design. The anti-proliferative, antioxidant, anti-inflammatory, and estrogenic activities of flavonoids could be harnessed to support classical treatments. While flavonoids show promising effects in cellular studies, the applied concentrations are almost impossible to reach with conventional food products. Changing the food processing might be one option to enhance the bioavailability. However, the bioavailability of flavonoids also depends on host factors like age and gender. Further research is required to understand the relevance of the nutrition in complex diseases.


### (6) Colours of Anthocyanins ###

> **_<a href="https://twitter.com/boas_pucker/status/1380415184741801984">TWEET</a>:_** Pelargonidin, cyanidin, peonidin, delphinidin, petunidin and malvidin are among the most common precursors of anthocyanins. #FlavonoidFriday (details: https://bit.ly/3d3BA5v)

<a href="https://twitter.com/boas_pucker/status/1380415184741801984">
<img alt="Colours of Anthocyanins (Tweet #6)" src="https://pbs.twimg.com/media/EygvdJdWEAoTWyK?format=jpg&name=900x900" width="40%" height="40%">
</a>


Pelargonidin (Pg), cyanidin (Cy), peonidin (Pn), delphinidin (Dp), petunidin (Pt) and malvidin (Mv) are the six most common anthocyanidins ([Sun et al., 2012](https://onlinelibrary.wiley.com/doi/full/10.1002/rcm.6209)). Anthocyanidins are unstable until sugar groups are attached. This modification step converts anthocyanidins (aglycones) into anthcyanins. Anthocyanins are contribute to a wide range of colours ranging from red to mangenta (cyanin), from orange to red (pelargonin), and from magenta to purple (delphinin) ([Ferreyra et al., 2012](https://doi.org/10.3389/fpls.2012.00222)).


### (7) Replacement of Anthocyanins by Betalains ###

> **_<a href="https://twitter.com/boas_pucker/status/1382951896089038849">TWEET</a>:_** While anthocyanins are present in most plant species, they are replaced by another group of pigments in some families of the Caryophyllales: betalains. #FlavonoidFriday
(src:https://doi.org/10.1111/nph.15980)

<a href="https://twitter.com/boas_pucker/status/1382951896089038849">
<img alt="Replacement of Anthocyanins by Betalains (Tweet #7)" src="https://pbs.twimg.com/media/EzEFv8lWEAAoA3H?format=jpg&name=large" width="30%" height="30%">
</a>


Both pigments (anthocyanins and betalains) were never observed together in the same plant which resulted in the mutual exclusion theory. A molecular mechanism which could cause this mutual exclusion is not known yet. Betalains do not occur in plant species outside the Caryophyllales. The current knowledge about the evolution of the betalain biosynthesis was recently reviewed in [Timoneda et al., 2019](https://doi.org/10.1111/nph.15980).


### (8) Anthocyanins - more than just colourful ###

> **_<a href="https://twitter.com/boas_pucker/status/1385473515101503489">TWEET</a>:_** Anthocyanins are not just colourful pigments, but have additional funtions: 1) metal-chelating agent 2) photoprotection & antioxidant 3) modulator of ROS signaling 4) delay of senescence #FlavonoidFriday (details: https://bit.ly/32Fvoup)

<a href="https://twitter.com/boas_pucker/status/1385473515101503489">
<img alt="Anthocyanins - more than just colourful (Tweet #8)" src="https://pbs.twimg.com/media/Ezou1oRWQAApjzb?format=png&name=900x900" width="30%" height="30%">
</a>


Anthocyanins are well known for the colouring of flowers and other plant tissue. However, the evolution of these pigments might not only be due to the colour. It is possible that additional functions provided evolutionary advantages. A publication by [Landi et al., 2015](https://doi.org/10.1016/j.envexpbot.2015.05.012) describes some of these functions:
1) Anthocyanins can serve as metal-chelating agents.
2) Anthocyanins can have a photoprotective and antioxidant effect.
3) The ROS signaling can be modulated by anthocyanins.
4) It seems that anthocyanins can delay senescencs that is promoted by nutient deficiency.
The functions and possible also additional ones can contribute differently to the evolutionary value of anthocyanins in numerous land plant species.


### (9) Red leaf colour of _Perilla frutescens_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1388002675455782912">TWEET</a>:_** A relative increase of anthocyanin biosynthesis over flavonol biosynthesis seems to explain the leaf colour difference between the Perilla frutescens cultivars JIZI1 and JIZI2. #FlavonoidFriday (details: https://bit.ly/3eAA1ve)


<a href="https://twitter.com/boas_pucker/status/1388002675455782912">
<img alt="Red leaf colour of _Perilla frutescens_ (Tweet #9)" src="https://pbs.twimg.com/media/E0MY1HHWQAgGKgN?format=png&name=900x900" width="30%" height="30%">
</a>


_Perilla frutescens_ is a Chinese plant with nutritional and medical use. Flavonoids are probably an important factor. The varieties JIZI1 and JIZI2 differ in the (relative) amount of accumulated flavonoids. The green vs. red leaf colour difference indicates that anthocyanins might be involved. Differential expression analysis revealed 15 enzymes involved in this pathway. [Jiang et al., 2020](https://doi.org/10.1038/s41598-020-73274-y) state that 109 flavones, 33 flavonols, 27 flavonoids, 22 anthocyanins, 20 flavanones, and 12 isoflavones differ in their concentration between both cultivars. A gene expression analysis of the flavonoid biosynthesis shows up-regulation of almost all steps except for CHI and FLS. This might suggest (1) a general up-regulation of the pathway and (2) that the flux is directed towards anthocyanins and away from flavonols.

However, several open questions remain:
1) Why are DFR and ANS not up-regulated on all parallel branches of the flavonoid biosynthesis?
2) Is UGT75C1 accepting flavonol derivates and not just anthocyanins?
3) Which sequences are the orthologs of well characterized transcriptional regulators like MYB75, TT8, and TTG1?
4) Is the transcriptome assembly available?



### (10) Decoration of anthocyanins differs between species ###

> **_<a href="https://twitter.com/boas_pucker/status/1390562044575850497">TWEET</a>:_** Anthocyanins are modified by glycosyltransferases, methyltransferases, and acyltransferases. These modifications can stabilize anthocyanins and are responsible for the huge diversity. #FlavonoidFriday (details: https://bit.ly/2QZPGww)

<a href="https://twitter.com/boas_pucker/status/1390562044575850497">
<img alt="Decoration of anthocyanins differs between species (Tweet #10)" src="https://pbs.twimg.com/media/E0xCO0CXEAEA9tq?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

[Tohge et al., 2017](https://doi.org/10.1093/jxb/erx177) provide a great summary of the current knowledge about the diversity of the flavonoid decoration/modification steps. While the aglycon biosynthesis is well characterized and highly conserved between species, the modification appears to be more flexible. Generally, sugars can be added at the OH groups at the C3, C5, and C7 positions. Not just single sugars, but also sugar chains or trees can serve as decoration. A recently identified flavonol modifying enzyme is BGLU6 ([Ishihara et al., 2016](https://doi.org/10.1093/jxb/erv546)) which shows differences between A. thaliana accessions i.e. within a species. Therefore, it is not surprising that these decoration pathways are different between plant species.



### (11) Repressors of the anthocyanin biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1393091204762402816">TWEET</a>:_** Repressors of the anthocyanin biosynthesis are regulated by (1) development, (2) environmental triggers, or (3) plant hormones. Proteins and small RNAs can play a role. #FlavonoidFriday (details:https://bit.ly/3w8cfhg)



<a href="https://twitter.com/boas_pucker/status/1390562044575850497">
<img alt="Repressors of the anthocyanin biosynthesis (Tweet #11)" src="https://pbs.twimg.com/media/E1U6b3VX0AAL3o1?format=jpg&name=4096x4096" width="30%" height="30%">
</a>


#### Negative regulators of the anthocyanin biosynthesis ####

An excellent Tansley review by [LaFountain & Yuan (2021)](https://doi.org/10.1111/nph.17397) summarizes the current knowlege about negative regulators of the anthocyanin biosynthesis and gives perspective for future research. Anthocyanin repressors are generally activated by (1) developmental processes, (2) environmental factors, or (3) plant hormones. Most repressors destablize the MBW complex which would otherwise activate anthocyanin biosynthesis genes. While the first anthocyanin repressors were identified more than 20 years ago, most were discovered in the last decade. Activators of the anthocyanin biosynthesis belong to R2R3-MYB subgroup 6 or 5. In contrast, many repressors fall in subgroup 4.
1) There are active repressors in subgroup 4, which can be classified into FaMYB1-type and AtMYB4-type. It appears that an EAR motif is crucial for the repressor function. LaFountain & Yuan describe the function of the repressors as a "break" instead of an "off-switch". 
2) Another group of repressors lost one MYB repeat and is characterized by the presence of a 'TLLLFR' motif. There are indications for multiple different origins of these anthocyanin biosynthesis repressors.
3) The CPC-clade of repressors emerged in the common ancestor of monocots and dicots. Members of this clade lack the bHLH interaction motif which leads to passive represssion by competing for the bHLH component of the MBW complex. Expression of these MYBs is activated by the MBW complex which creates a periodic pattern of gene expression which becomes visible as spots or stripes of pigmentation.
4) miR156-SPL is highly expressed in seedlings and down-regulated a negative regulator (_SQUAMOSA PROMOTER BINDING PROTEIN-LIKE 9_) of the anthocyanin biosynthesis. This explains a temporal pigmentation of seedlings called "juvenile reddening" which is lost in later stages.
5) miR858 targets the anthocanin activator MYB113 in A. thliana. miR828 targets the negative regulator _MYBL2_ in _A. thaliana_, but the positive regulator _MYB7-like_ in tomato.
6) GL2 in _A. thaliana_ is a HD-ZIP family member that down-regulates _PAP2_ and _MYB113_. HAT1 is another HD-ZIP member that represses anthocyanin biosynthesis. GL2 and HAT1 might be a lineage-specific specialty of _A. thaliana_. In contrast, HB1 (discovered in apple) might be a more conserved repressor of the HD-ZIP family.


### (12) MYB117 can boost the anthocyanin biosynthesis in poplar ###

> **_<a href="https://twitter.com/boas_pucker/status/1395620374864302083">TWEET</a>:_** Overexpression of poplar MYB117 boosts the anthocyanin biosynthesis. Up-regulation of the flavonoid 3',5'-hydroxylase genes leads to increased B-ring hydroxylation of flavonoids. #FlavonoidFriday (details: https://bit.ly/3wlSlzv)

<a href="https://twitter.com/boas_pucker/status/1395620374864302083">
<img alt="MYB117 can boost the anthocyanin biosynthesis in poplar (Tweet #12)" src="https://pbs.twimg.com/media/E144mTUXEAI8rij?format=jpg&name=medium" width="30%" height="30%">
</a>

[Ma et al., 2021](https://academic.oup.com/jxb/article/72/10/3864/6169232?login=true) present the poplar MYB117 as a powerful activator of the anthocyanin biosynthesis. Overexpression under control of the 35S promoters provides a general boost to the anthocyanin biosynthesis. The up-regulation of F3'5'H genes causes an increased B-ring hydroxylation which affects also other branches of the flavonoid biosynthesis. Multiple investigations of several flavonoid biosynthesis activators suggest that different MYBs can fine tune the hydroxylation level of flavonoids. MYB117 sits in an array of six PAP1-type MYBs: MYB113, MYB116, MYB117, MYB118, MYB119, and MYB120.


### (13) Tobacco CHS is crucial for salt tolerance ###

> **_<a href="https://twitter.com/boas_pucker/status/1398172184602943491">TWEET</a>:_** Overexpression of the tobacco MYB4 reduces the salt tolerance of seedlings through repression of flavonoid production. The flavonoid biosynthesis is inhibited by down-regulation of the CHS (first step) #FlavonoidFriday (details: https://bit.ly/2TfMcqm)


<a href="https://twitter.com/boas_pucker/status/1398172184602943491">
<img alt="Tobacco CHS is crucial for salt tolerance (Tweet #13)" src="https://pbs.twimg.com/media/E2dFaAeXoAAUdjK?format=png&name=small" width="30%" height="30%">
</a>

Salt stress is known to induce the flavonoid biosynthesis. [Chen et al. 2019](https://doi.org/10.3389/fpls.2019.00178) report that rutin content increased in tobacco under salt stress. However, the overexpression of NtMYB4 caused a reduced production of rutin leading to a lower salt tolerance. MYB4 overexpression lines were more similar to the WT when rutin was externally supplied. Rutin is probably involved in reduction of ROS which accumulates under salt stress. It seems that NtMYB is a strong negative regulator of the CHS, but does not affect ANS substantially. However, down-regulation of CHS by RNAi lead to a reduced transcript abundance of the downstream genes.


### (14) TTG2 controls epicatechin transport ###

> **_<a href="https://twitter.com/boas_pucker/status/1400693803057438724">TWEET</a>:_** TTG2 (WRKY44) influences the seed coat pigmentation through regulation of the epicatechin transport (TT12, AHA10). Formation of a MBW-like complex comprising TTG2+TTG1+TT8 was postulated #FlavonoidFriday (details: https://bit.ly/3wPIK4a)


<a href="https://twitter.com/boas_pucker/status/1400693803057438724">
<img alt="TTG2 controls epicatechin transport (Tweet #14)" src="https://pbs.twimg.com/media/E2xycrXWQAEfcLg?format=png&name=900x900" width="30%" height="30%">
</a>

[Gonzalez et al., 2016](https://doi.org/10.1016/j.ydbio.2016.03.031) present TTG2 as an important regulator of epicatechin transporters like TT12/MATE and TT13/AHA10. Epicatechins are imported into the vacuoles of seed coats where they are condensed into proanthocyanidins (condensed tannins). Proanthocyanidins (PAs) provide the dark colouration to seeds. There is a large mutant collection comprising lines with specific blocks in the PA biosynthesis or accumulation pathway. These are called transparent testa (tt) mutants [Appelhagen et al., 2014](https://doi.org/10.1007/s00425-014-2088-0). A transcription factor complex of MYB+bHLH+WDR is known to regulate several steps in the flavonoid biosynthesis and especially in the anthocyanin and PA biosynthesis. _TTG2_ (_WRKY44_) expression is controlled by the MBW complex [Ishida et al., 2007](https://doi.org/10.1105/tpc.107.052274). TTG2 shows high binding affinity towards the W box, a motif of (C/T)TGAC(T/C), which was detected in the promoter sequences of many target genes. TTG2 is specific to the PA biosynthesis, because anthocyanins are not affected in the ttg2 mutant. The TT12 promoter region contains several W boxes, but it is not cleary which of them is the actual TTG2 binding site. TT12 overexpression partialy rescued the ttg2 phenotype thus suggesting that this is one of the most important targets. In summary, TTG2 appears to be a regulator of the PA transport pathway.


### (15) Role of TT10 in the seed coloration ###

> **_<a href="https://twitter.com/boas_pucker/status/1403230516552237056">TWEET</a>:_** TT10 is a laccase-like polyphenol oxidase which appears to be involved in the oxidative polymerization of flavonoids in developing Arabidopsis thaliana seed coats. The TT10 products are responsible for the brown seed color. #FlavonoidFriday (details: https://bit.ly/34KmA7u)


<a href="https://twitter.com/boas_pucker/status/1403230516552237056">
<img alt="Role of TT10 in the seed coloration (Tweet #15)" src="https://pbs.twimg.com/media/E286633XoAQxJZC?format=jpg&name=large" width="30%" height="30%">
</a>


[Pourcel et al., 2005](https://dx.doi.org/10.1105%2Ftpc.105.035154) characterized _TRANSPARENT TESTA 10_ (_TT10_) which encodes a laccase-like enzymes which is most likely involved in the oxidative polymerization of epicatechins and catechins in the seed coat of _A. thaliana_. These oxidized compounds provide the brown coloration of seeds. Oxidized forms of colorless epicatechins and catechins are called quinones which turn from yellow to brown with increase in oxidation level. Given the TT10 function, it is not surprising that the _tt10_ mutant shows a delayed appearance of the brown color. None of the other 16 laccase-like proteins in _A. thaliana_ are able to fullfill the TT10 function. Instead an autoxidation is assumed in the _tt10_ mutant. Epicatechin and catechin accumulate in the vacuole, but might be released into the cytoplasm upon cell death through a burst of the vacuole. TT10 would get in contact with its substrates and could produce oxidized and polymerized products. Additionally, TT10 might process PAs in the cell wall. TT10 is also involved in the formation of flavonol dimers (quercetine rhamnoside oxidative polymerization), but this product does not contribute to the brown seed color. Interestingly, it appears that _TT10_ is not regulated by the TT2+TT8+TTG1 complex like other genes in the proanthocyanidin biosynthesis.



### (16) Phenylpropanoids in strawberry ###

> **_<a href="https://twitter.com/boas_pucker/status/1405767234367234048">TWEET</a>:_** F3'H seems to be a decisive step in the flavonoid biosynthesis of strawberry, because higher gene expression correlates with increased content of specific flavonoids #FlavonoidFriday (details: https://bit.ly/3vw4VeZ)


<a href="https://twitter.com/boas_pucker/status/1405767234367234048">
<img alt="Phenylpropanoids in strawberry (Tweet #16)" src="https://pbs.twimg.com/media/E4IpNXlVkAAfp2b?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

Flavonoids have nutritional benefits for humans mostly due to their antioxidant activity. [Pott et al., 2020](https://www.nature.com/articles/s41598-020-76946-x) investigated the flavonoid biosynthesis as well as the general phenylpropanoid biosynthesis in strawberry. The composition and concentration of flavonoids changes during strawberry development: tannins are produced in the early stages and flavonols and anthocyanins in the later stages. Important for the pigmentation of the fruits are glycosylated pelargonidins and cyanidins. The authors investigated the heritability of phenylpropanoid metabolites including flavonoids. QTL for phenylpropanoids were identified, but there is a large number of QTL spanning substantial parts of almost all linkage groups. However, there are hotspots which seem to be associated with multiple phenlypropanoids. An interesting observation is that one copy of the anthocyanin repressor FaMYB was located in a QTL, but the gene expression does not correlate with the investigated flavonoids. The authors present F3'H as a potentially decisive step in the flavonoid biosynthesis that can control the metabolic flux. The correlation of F3'H gene expression correlates with the amount of specific flavonoiods (see paper for details). The substrate specificity of enzymes with respect to hydroxylation patterns needs further investigation.


### (17) MYB4, MYB7, and MYB32 are anthocyanin repressors ###

> **_<a href="https://twitter.com/boas_pucker/status/1408334154614247425">TWEET</a>:_** MED5a & MED5b control the expression of AtMYB4 which interacts with the bHLHs TT8, GL3, and EGL3 leading to a reduced expression of anthocyanin and proanthocyanidin biosynthesis genes #FlavonoidFriday (details: https://bit.ly/3qqsMM1)


<a href="https://twitter.com/boas_pucker/status/1408334154614247425">
<img alt="MYB4, MYB7, and MYB32 are anthocyanin repressors (Tweet #17)" src="https://pbs.twimg.com/media/E4tj1ieX0AU-4cw?format=jpg&name=large" width="40%" height="40%">
</a>


[Wang et al., 2019](https://doi.org/10.1111/tpj.14570) report that MYB4, MYB7, and MYB32 interact with the bHLHs TT8, GL3, and EGL3 leading to a repression of the flavonoid biosynthesis. MYB4 is also a direct repressor of Arogenate Dehydratase 6 (ADT6) - the last step in the shikimate biosynthesis, which feeds substrate to the flavonoid biosynthesis. A complex of MED5a and MED5b regulates the gene expression of MYB4, MYB7, and MYB32. If only MED5a is knocked-out, the anthocyanin accumulation is increased due to a reduced expression of the repressors MYB4, MYB7, and MYB32. While MYB4, MYB7, and MYB32 can bind to bHLHs, they do not interact with the anthoyanin activating MYBs. MYB4, MYB7, and MYB32 show different affinities for specific bHLHs suggesting a fine regulation of specific MBW complexes.



### (18) Flavonoid biosynthesis regulator miR858 ###

> **_<a href="https://twitter.com/boas_pucker/status/1410855763678011398">TWEET</a>:_** pri-miR858a encodes a small peptide which promotes miR858 transcription. miR858a controls lignin and flavonoid biosynthesis through repression of MYB12 #FlavonoidFriday (details: https://bit.ly/3hl2rL6)

<a href="https://twitter.com/boas_pucker/status/1410855763678011398">
<img alt="Flavonoid biosynthesis regulator miR858 (Tweet #18)" src="https://pbs.twimg.com/media/E5RYxV0WQAEHYLN?format=jpg&name=large" width="40%" height="40%">
</a>

miRNAs are single stranded RNAs that regulate gene expression through increased transcript degradation or repressed translation. These miRNAs can encode small proteins which increase their own transcription. miR858 is a repressor of the MYB11/MYB12/MYB111 transcription. This negative regulation results in reduced flavonol production. [Sharma et al., 2016](https://pubmed.ncbi.nlm.nih.gov/27208307/), [Wang et al., 2016](https://pubmed.ncbi.nlm.nih.gov/27450422/), and [Sharma et al., 2020](https://www.nature.com/articles/s41477-020-00769-x) investigated the functions of miR858. The latest study used a modification via (1) CRISPR/Cas-based modification and (2) supplementation of miPEP858a.
miPEP858a was identified through the test of all potential ORFs in a GUS fusion experiment. Only one of the potential ORFs was actually translated leading to detectable GUS activity.
Supplementation of miPEP585a in the media resulted in enhanced transcription of miR858 and slightly shorter roots. Several additional experiments support that miPEP858a is important in the transcriptional control of miR858.
Overexpression of miR858 resulted in reduced flavonol and anthocyanin aglycones as well as increased lignin levels. The modification (knock-out) of miR858 resulted in the opposite effect i.e. increased anthocyanin and flavonol levels at the cost of lignin.
It is known that flavonoids have a negative impact on auxin transportation and plant growth. Loss of miR858 leads to lower expression of auxin-associated genes, while the overexpression of miR858 increases the expression of these genes.


### (19) Anthocyanin degradation ###

> **_<a href="https://twitter.com/boas_pucker/status/1413384926473379840">TWEET</a>:_** Accumulation of anthocyanins in the central vacuole leads to a coloration of plant structures. Numerous modifications and degradation determine how quickly the color fades #FlavonoidFriday (details: https://bit.ly/3yCpurZ)


<a href="https://twitter.com/boas_pucker/status/1413384926473379840">
<img alt="Anthocyanin degradation (Tweet #19)" src="https://pbs.twimg.com/media/E50YKbgWUAAE8DW?format=jpg&name=medium" width="40%" height="40%">
</a>


[Passeri et al., 2016](https://doi.org/10.3389/fpls.2016.00153) reviewed the stabilization of anthocyanins in the vacuole with respect to biotechnological applications. Enriching food with anthocyanins is important, because several studies identified health promoting effects of anthocyanins. Generating new colors through modifications of anthocyanins is of huge importance for the ornamental plant and cut flower industry.
Anthocyanin decorating enzymes include UDP-glucose:flavonoid 3-O-glucosyltransferase (3UFGT), rhamnosyl transferase (RT), UDP-glucose:flavonoid 5-O-glucosyltransferase (5UFGT), anthocyanin acyltransferase (AAT), and methyltransferase (MT). Expression of the corresponding genes is controlled by the MBW complex. Loss of anthocyanin pigmentation is often caused by mutations in the regulator genes (MYB, bHLH, WDR), because mutations in structural genes would be insufficient or could have pleitropic effects.
Active degradation of anthocyanins could be catalyzed by polyphenol oxidases if the glycosylation is missing/lost. The relevance of different anthocyanin decorations on degrading enzymes is still under investigation. This anthocyanin degradation can strongly affect the market value of ornamental plants. Therefore, it is important to understand the process and to improve plants bassed on this knowledge. A vacuolar peroxidase is suspected to be involved in this anthocyanin degradation process. A proton pump (AHA10) influences the pH value of the vacuole, but this has only a minor effect on anthocyanin fading if any. The anthocyanin degradation in leaves might be beneficial for photosynthesis efficiency. The degradation in flowers is coupled with the release of fragrant volatils which could be a signal for pollinators.



### (20) Mutual exclusion of anthocyanins and betalains in _Hylocereus_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1415921640060297219">TWEET</a>:_** Anthocyanins are replaced by betalains in some Caryophyllales. There is a mutual exclusion of both pigments. A species is only anthocyanin-pigmented OR betalain-pigmented #FlavonoidFriday (https://bit.ly/3xGuRXf & https://rdcu.be/cn0UW)

<a href="https://twitter.com/boas_pucker/status/1415921640060297219">
<img alt="Mutual exclusion of anthocyanins and betalains in _Hylocereus_ (Tweet #20)" src="https://pbs.twimg.com/media/E50laJ-XsAMoet0?format=jpg&name=medium" width="40%" height="40%">
</a>

A recent publication by Fan et al., 2020 in BMC Plant Biology claimed the detection of anthocyanins in the betalain-pigmented genus _Hylocereus_ and stated that anthocyanins are responsible for the red coloration of pitaya fruits. This analysis was based on the comparison of a white, a pink, and a red pitaya cultivar. However, this paper was retracted due to various issues. A [re-analysis](https://doi.org/10.1186/s12870-021-03080-9) of the RNA-seq data sets revealed that transcripts of the anthocyanin biosynthesis genes (_DFR_ & _ANS_) are barely detectable. Moreover, the abundances of these transcripts in different cultivars do not support the corresponding pigment phenotypes. Several studies already reported that betalains are responsible for the red coloration of pitaya fruits. In summary, the re-analysis supports the mutual exclusion paradigm and suggests that not anthocyanin, but betalains, are responsible for the coloration of pitaya fruits.



### (21) KIPEs - automatic identification of flavonoid biosynthesis genes or enzymes ###

> **_<a href="https://twitter.com/boas_pucker/status/1418443256405262345">TWEET</a>:_** Looking for flavonoid biosynthesis genes/enzymes in a transcriptome or genome assembly? KIPEs can perform this identification automatically in your species of interest #FlavonoidFriday (https://bit.ly/3fpykTw & https://bit.ly/3ikpFlh)

<a href="https://twitter.com/boas_pucker/status/1418443256405262345">
<img alt="KIPEs - automatic identification of flavonoid biosynthesis genes or enzymes (Tweet #21)" src="https://pbs.twimg.com/media/Eu4yOCfWYAIMKxJ?format=jpg&name=4096x4096" width="20%" height="20%">
</a>

[KIPEs](https://github.com/bpucker/KIPEs) is freely available via github. Data sets are included to allow the identification of genes/enzymes of the core flavonoid biosynthesis pathway. Additional data sets to find the players of additional reactions are under construction. More pathways will be supported in the future. Please get in touch if you have suggestions for pathways that should be included.



### (22) Brassinosteroids inhibit flavonoid biosynthesis in apple ###

> **_<a href="https://twitter.com/boas_pucker/status/1421017725330149377">TWEET</a>:_** Brassinosteroid treatment reduces the flavonoid biosynthesis in apple. BR-induced MdBEH2.2 might form a complex with the anthocyanin regulator MdMYB60 leading to this inhibition #FlavonoidFriday (details: https://bit.ly/3x7GVQ4)

<a href="https://twitter.com/boas_pucker/status/1421017725330149377">
<img alt="Brassinosteroids inhibit flavonoid biosynthesis in apple (Tweet #22)" src="https://pbs.twimg.com/media/E7hyOQPXMAE8pLf?format=jpg&name=medium" width="20%" height="20%">
</a>

[Wang et al., 2021](https://doi.org/10.1093/jxb/erab284) investigated the inhibitation of the flavonoid biosynthesis in red-fleshed apple by brassinosteroids. Brassinosteroids (BRs) are steroid hormones with an inibitory effect on the flavonoid biosynthesis. A central inhibitor involved in this process is probably MdBEH2.2 which form a coplex with MdMYB60 leading to an inhibition of this MYB protein. However, many additional proteins are involves in detecting BRs and transmitting the triggered singnal. While treatments with BRs lead to a reduced flavonoid accumulation, the opposite effect was observed in treatments with a BR biosynthesis inhibitor (brassinazole).


### (23) Auxiliary proteins in the flavonoid biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1423531785556664322">TWEET</a>:_** Chalcone isomerase-like (CHIL) and pathogenesis-related 10 (PR10) proteins play probably non-enzymatic, but important regulatory roles in the flavonoid biosynthesis #FlavonoidFriday (details: https://bit.ly/3fBuPsw)

<a href="https://twitter.com/boas_pucker/status/1423531785556664322">
<img alt="Auxiliary proteins in the flavonoid biosynthesis (Tweet #23)" src="https://pbs.twimg.com/media/E8Fjt8NWYAAELgq?format=jpg&name=4096x4096" width="20%" height="20%">
</a>


Two families of auxiliary proteins are important regulators of the flavonoid biosynthesis ([Dastmalchi, 2021](https://onlinelibrary.wiley.com/doi/epdf/10.1111/tpj.15446)):
(1) Chalcone isomerase-like (CHIL) proteins bind to CHS and influence the catalyzed reaction. This might be necessary to reduce the promiscuity of CHS, which was observed in _in vitro_ assays. CHILs increased the flavonoid biosynthesis in plants. Loss of this activity can lead to a loss of pigmentation in flower and fruit. These CHILs are closely related to _bona fide_ CHIs and form the type IV CHI clade. Members of the type III CHIs have also been reported to act as polyketide-binding proteins (PBPs). These non-CHI proteins lack the amino acids of the CHI active center and position the ligand naringenin in a non-productive. However, it appears that CHILs are crucial for the coordination between CHS and CHI. It is possible that CHILs influence the biosynthesis of different anthocyanins, but do not alter the overall amount. The R2R3-MYB TT2 was identified as regulator of _A. thaliana_ CHIL.

(2) Pathogenesis-related 10 (PR10) proteins seem to bind intermediates of the flavonoid biosynthesis and could have an impact on the anthocyanin composition. PR10 is also involved in other spececialized metabolism pathways. Unfortunately, there is no uniform nomenclature for members of this gene family, because they were discovered and described independently in different species. PR1 in _Petroselium crispum_, _Bet v 1_ in _Betula verrucosa_, and Major latex proteins (MLPs) in _Papaver somniferum_. The PR10 clade is highly susceptible to neofunctionalization.


### (24) Flavonoid absorption spectra ###

> **_<a href="https://twitter.com/boas_pucker/status/1426076051352875013">TWEET</a>:_** Flavonoid absorption spectra depend on the pH. Cyanidin- and pelargonidin-based anthocyanins peak in blue-green wavelength at low pH, but shift to longer wavelengths at higher pH #FlavonoidFriday (details: https://bit.ly/37F7PVa)

<a href="https://twitter.com/boas_pucker/status/1426076051352875013">
<img alt="Flavonoid absorption spectra (Tweet #24)" src="https://pbs.twimg.com/media/E8pkBfmWEAQvU1p?format=jpg&name=large" width="20%" height="20%">
</a>


[Stavenga et al., 2021](https://doi.org/10.3389/fpls.2020.600124) investigated the impact of pH change on the absorption spectra of extracts from flowers. The color of flavonoids accumulating in the central vacuole is generally dependent on the vacuolar pH. Investigated species are the red flowering _Papaver dubium_, the orange flowering _Meconopsis cambrica_, and two _Mandevilla sanderi_ varities that flower in red and white, respectively. The absorption spectra of red extracts from _P. dubium_ flowers can be explained by cyanidin and pelargonidin glycosides. An analysis of _P. rhoeas_ extracts revealed that the contribution of flavonols with the same hydroxylation pattern (kaempferol and quercetin) is rather small. White flowers of _M. sanderi_ show a strong absorption in the UV range indicating the presence of flavonols. Methanolic extracts represent the native plant tissue well. 



### (25) Petal color differences in Lysimachia arvensis ###

> **_<a href="https://github.com/bpucker/FlavonoidFriday">TWEET</a>:_** DFR and F3'5'H are two important genes influencing the petal color of Lysimachia arvensis. DFR copies show amino acid differences which probably altered their substrate affinity #FlavonoidFriday (details: )



[Sanchez-Cabera et al.](https://www.frontiersin.org/articles/10.3389/fpls.2021.633979/full) investigated the flower color transition from blue to orange in _Lysimachia arvensis_ based on a transcriptomic data set. Blue and orange flowered L. arvenssis plants can be found across Europe. Interestingly, the proportion of blue-flowered plants increases with temperature and daylength, while higher precipitation leads to a decrease. 39 of all 140 genes associated with the flavonoid biosynthesis showed differential expression. F3'5'H and DFR were among these DEGs. F3'5'H showed reduced expression in orange petals and two DFR copies revealed differential expression. These DFR copies showed amino acid substitutions that affected the substrate specificity region: primarily malvidin (blue) to primarily pelarognidin (orange). A phylogenetic analysis revealed that these DFR copies belong to two distinct lineages that resulted from a duplication deep in the angiosperms. Another important factor for the orange coloration is probably the high abundance of BZ1-2 which adds a glucose to the 3-position of anthocyanidins. Blue petals showed high abundance of OMT (methylation of delphinidin) and 3GT (adds rhamnoside), which are probably required for stabilization of the color. 




## Contact ##
Do you have a recent open access publication about the flavonoid biosynthesis that could be featured here? Please get in touch: [Dr. Boas Pucker (email)](mailto:bpucker@cebitec.uni-bielefeld.de?subject=[GitHub]FlavonoidFriday)


## References ##

<details>
<summary>(click to expand)</summary>
<p>
Pucker, B., Reiher, F. and Schilbert, H. M. Automatic identification of players in the flavonoid biosynthesis with application on the biomedicinal plant Croton tiglium. Plants 2020, 9, 1103. doi:[10.3390/plants9091103](https://www.mdpi.com/2223-7747/9/9/1103/htm).

Timoneda, A., Feng, T., Sheehan, H., Walker-Hale, N., Pucker, B., Lopez-Nieves, S., Guo, R., Brockington, S. (2019). The evolution of betalain biosynthesis in Caryophyllales. New Phytologist. doi:[10.1111/nph.15980](https://doi.org/10.1111/nph.15980)

Susanna Pollastri, Massimiliano Tattini, Flavonols: old compounds for old roles, Annals of Botany, Volume 108, Issue 7, November 2011, Pages 1225–1233, doi:[10.1093/aob/mcr234](https://doi.org/10.1093/aob/mcr234)

Landi, M. et al. "Multiple functional roles of anthocyanins in plant-environment interactions." Environmental and Experimental Botany 119 (2015): 4-17.

Ahn-Jarvis, J.H.; Parihar, A.; Doseff, A.I. Dietary Flavonoids for Immunoregulation and Cancer: Food Design for Targeting Disease. Antioxidants 2019, 8, 202. doi:[10.3390/antiox8070202](https://doi.org/10.3390/antiox8070202)

Sun, J., Lin, L.‐z. and Chen, P. (2012), Study of the mass spectrometric behaviors of anthocyanins in negative ionization mode and its applications for characterization of anthocyanins and non‐anthocyanin polyphenols. Rapid Commun. Mass Spectrom., 26: 1123-1133. doi:[10.1002/rcm.6209](https://doi.org/10.1002/rcm.6209)
</p>
</details>

